{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Extracting annotations from Pubtator\n",
    "\n",
    "There are three types of parameters that one can customize in the query:\n",
    "Bioconcept: PubTator includes annotations for five bioconcepts, i.e., Gene, Disease, Chemical, Species, Mutation and BioConcept (All types are included)\n",
    "PMIDs: single pmid or a pmid list (e.g. pmid=23819905,23819906)\n",
    "Format: PubTator, BioC, and PubAnnotation(JSON)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['16013892', '17919318', '18315855', '19187544', '19568424', '19635124', '19682364', '20015400', '21408132', '21450075', '22726254', '23028498', '23074659', '23110173', '23193411', '22590978', '23062055', '23284733', '23202545', '22906110', '22960954', '23351276', '23710205', '23936115', '24223640', '24278493', '24278022', '24586153', '24638205', '24816104', '24824328', '24895650', '27713263', '24875882', '24884954', '24916787', '24968358', '25098254', '25422070', '25470665', '25856575', '25948687', '26041555', '26325027', '26266418', '26678835', '26696249', '26684827', '26884414', '27094905', '26961383', '27328634', '27257056', '27406444', '27381218', '27708643', '27677239', '27572397', '28229049', '28346512', '28540291', '28465619', '28526080', '28588292', '28805676', '28720711', '28806134', '29159184', '29263859', '29302259', '29426899', '29458350', '29497534', '29656631', '29907804', '29933625', '30157967', '30260954', '30282716', '21160452', '30558354', '30704498', '30742627', '30813403', '30857374', '31013925', '31052199', '31324182', '31340782', '31720054', '31703426', '31898474', '31817829', '31847268', '31671826', '32079545', '32206052']\n"
     ]
    }
   ],
   "source": [
    "pmid = []\n",
    "with open('D:\\\\Lab\\\\Projects\\\\corona\\\\100_pmid.txt','r') as list:\n",
    "    for line in list:\n",
    "        line = line.strip()\n",
    "        pmid.append(line)\n",
    "print(pmid)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/16013892/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/17919318/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/18315855/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/19187544/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/19568424/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/19635124/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/19682364/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/20015400/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/21408132/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/21450075/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/22726254/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/23028498/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/23074659/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/23110173/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/23193411/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/22590978/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/23062055/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/23284733/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/23202545/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/22906110/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/22960954/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/23351276/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/23710205/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/23936115/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/24223640/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/24278493/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/24278022/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/24586153/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/24638205/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/24816104/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/24824328/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/24895650/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/27713263/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/24875882/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/24884954/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/24916787/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/24968358/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/25098254/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/25422070/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/25470665/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/25856575/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/25948687/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/26041555/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/26325027/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/26266418/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/26678835/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/26696249/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/26684827/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/26884414/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/27094905/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/26961383/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/27328634/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/27257056/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/27406444/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/27381218/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/27708643/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/27677239/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/27572397/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/28229049/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/28346512/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/28540291/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/28465619/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/28526080/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/28588292/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/28805676/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/28720711/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/28806134/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/29159184/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/29263859/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/29302259/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/29426899/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/29458350/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/29497534/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/29656631/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/29907804/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/29933625/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/30157967/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/30260954/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/30282716/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/21160452/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/30558354/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/30704498/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/30742627/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/30813403/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/30857374/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/31013925/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/31052199/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/31324182/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/31340782/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/31720054/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/31703426/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/31898474/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/31817829/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/31847268/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/31671826/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/32079545/PubAnnotation/\n",
      "https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/32206052/PubAnnotation/\n"
     ]
    }
   ],
   "source": [
    "url_list = []\n",
    "for id in pmid:\n",
    "    url = 'https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/'+id+'/PubAnnotation/'\n",
    "    url_list.append(url)\n",
    "for line in url_list:\n",
    "    print(line)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<Response [200]>\n"
     ]
    }
   ],
   "source": [
    "item = requests.get('https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/32206052/PubAnnotation/')\n",
    "print(item)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'sourceid': '16013892', 'sourcedb': 'PubMed', 'text': 'Appropriate models for the management of infectious diseases. BACKGROUND: Mathematical models have become invaluable management tools for epidemiologists, both shedding light on the mechanisms underlying observed dynamics as well as making quantitative predictions on the effectiveness of different control measures. Here, we explain how substantial biases are introduced by two important, yet largely ignored, assumptions at the core of the vast majority of such models. METHODS AND FINDINGS: First, we use analytical methods to show that (i) ignoring the latent period or (ii) making the common assumption of exponentially distributed latent and infectious periods (when including the latent period) always results in underestimating the basic reproductive ratio of an infection from outbreak data. We then proceed to illustrate these points by fitting epidemic models to data from an influenza outbreak. Finally, we document how such unrealistic a priori assumptions concerning model structure give rise to systematically overoptimistic predictions on the outcome of potential management options. CONCLUSION: This work aims to highlight that, when developing models for public health use, we need to pay careful attention to the intrinsic assumptions embedded within classical frameworks.', 'denotations': [{'obj': 'Disease:D003141', 'span': {'begin': 41, 'end': 60}}, {'obj': 'Disease:D007251', 'span': {'begin': 887, 'end': 896}}]}]\n",
      "[{'sourceid': '17919318', 'sourcedb': 'PubMed', 'text': \"Modeling and detection of respiratory-related outbreak signatures. BACKGROUND: Time series methods are commonly used to detect disease outbreak signatures (e.g., signals due to influenza outbreaks and anthrax attacks) from varying respiratory-related diagnostic or syndromic data sources. Typically this involves two components: (i) Using time series methods to model the baseline background distribution (the time series process that is assumed to contain no outbreak signatures), (ii) Detecting outbreak signatures using filter-based time series methods. METHODS: We consider time series models for chest radiograph data obtained from Midwest children's emergency departments. These models incorporate available covariate information such as patient visit counts and smoothed ambient temperature series, as well as time series dependencies on daily and weekly seasonal scales. Respiratory-related outbreak signature detection is based on filtering the one-step-ahead prediction errors obtained from the time series models for the respiratory-complaint background. RESULTS: Using simulation experiments based on a stochastic model for an anthrax attack, we illustrate the effect of the choice of filter and the statistical models upon radiograph-attributed outbreak signature detection. CONCLUSION: We demonstrate the importance of using seasonal autoregressive integrated average time series models (SARIMA) with covariates in the modeling of respiratory-related time series data. We find some homogeneity in the time series models for the respiratory-complaint backgrounds across the Midwest emergency departments studied. Our simulations show that the balance between specificity, sensitivity, and timeliness to detect an outbreak signature differs by the emergency department and the choice of filter. The linear and exponential filters provide a good balance.\", 'denotations': [{'obj': 'Species:1392', 'span': {'begin': 201, 'end': 208}}, {'obj': 'Species:9606', 'span': {'begin': 645, 'end': 653}}, {'obj': 'Species:9606', 'span': {'begin': 744, 'end': 751}}, {'obj': 'Species:1392', 'span': {'begin': 1139, 'end': 1146}}]}]\n",
      "[{'sourceid': '18315855', 'sourcedb': 'PubMed', 'text': \"Comparative phyloinformatics of virus genes at micro and macro levels in a distributed computing environment. BACKGROUND: Preparedness for a possible global pandemic caused by viruses such as the highly pathogenic influenza A subtype H5N1 has become a global priority. In particular, it is critical to monitor the appearance of any new emerging subtypes. Comparative phyloinformatics can be used to monitor, analyze, and possibly predict the evolution of viruses. However, in order to utilize the full functionality of available analysis packages for large-scale phyloinformatics studies, a team of computer scientists, biostatisticians and virologists is needed--a requirement which cannot be fulfilled in many cases. Furthermore, the time complexities of many algorithms involved leads to prohibitive runtimes on sequential computer platforms. This has so far hindered the use of comparative phyloinformatics as a commonly applied tool in this area. RESULTS: In this paper the graphical-oriented workflow design system called Quascade and its efficient usage for comparative phyloinformatics are presented. In particular, we focus on how this task can be effectively performed in a distributed computing environment. As a proof of concept, the designed workflows are used for the phylogenetic analysis of neuraminidase of H5N1 isolates (micro level) and influenza viruses (macro level). The results of this paper are hence twofold. Firstly, this paper demonstrates the usefulness of a graphical user interface system to design and execute complex distributed workflows for large-scale phyloinformatics studies of virus genes. Secondly, the analysis of neuraminidase on different levels of complexity provides valuable insights of this virus's tendency for geographical based clustering in the phylogenetic tree and also shows the importance of glycan sites in its molecular evolution. CONCLUSION: The current study demonstrates the efficiency and utility of workflow systems providing a biologist friendly approach to complex biological dataset analysis using high performance computing. In particular, the utility of the platform Quascade for deploying distributed and parallelized versions of a variety of computationally intensive phylogenetic algorithms has been shown. Secondly, the analysis of the utilized H5N1 neuraminidase datasets at macro and micro levels has clearly indicated a pattern of spatial clustering of the H5N1 viral isolates based on geographical distribution rather than temporal or host range based clustering.\", 'denotations': [{'obj': 'Disease:D007251', 'span': {'begin': 214, 'end': 238}}, {'obj': 'Disease:D007251', 'span': {'begin': 1356, 'end': 1365}}, {'obj': 'Gene:4758', 'span': {'begin': 1307, 'end': 1320}}, {'obj': 'Gene:4758', 'span': {'begin': 1654, 'end': 1667}}, {'obj': 'Gene:4758', 'span': {'begin': 2320, 'end': 2333}}]}]\n",
      "[{'sourceid': '19187544', 'sourcedb': 'PubMed', 'text': \"Detection of genetically modified organisms (GMOs) using isothermal amplification of target DNA sequences. BACKGROUND: The most common method of GMO detection is based upon the amplification of GMO-specific DNA amplicons using the polymerase chain reaction (PCR). Here we have applied the loop-mediated isothermal amplification (LAMP) method to amplify GMO-related DNA sequences, 'internal' commonly-used motifs for controlling transgene expression and event-specific (plant-transgene) junctions. RESULTS: We have tested the specificity and sensitivity of the technique for use in GMO studies. Results show that detection of 0.01% GMO in equivalent background DNA was possible and dilutions of template suggest that detection from single copies of the template may be possible using LAMP. CONCLUSION: This work shows that GMO detection can be carried out using LAMP for routine screening as well as for specific events detection. Moreover, the sensitivity and ability to amplify targets, even with a high background of DNA, here demonstrated, highlights the advantages of this isothermal amplification when applied for GMO detection.\", 'denotations': []}]\n",
      "[{'sourceid': '19568424', 'sourcedb': 'PubMed', 'text': \"Lactate dehydrogenase-elevating virus induces systemic lymphocyte activation via TLR7-dependent IFNalpha responses by plasmacytoid dendritic cells. BACKGROUND: Lactate dehydrogenase-elevating virus (LDV) is a natural infectious agent of mice. Like several other viruses, LDV causes widespread and very rapid but transient activation of both B cells and T cells in lymphoid tissues and the blood. The mechanism of this activation has not been fully described and is the focus of the current studies. PRINCIPAL FINDINGS: A known inducer of early lymphocyte activation is IFNalpha, a cytokine strongly induced by LDV infection. Neutralization of IFNalpha in the plasma from infected mice ablated its ability to activate lymphocytes in vitro. Since the primary source of virus-induced IFNalpha in vivo is often plasmacytoid dendritic cells (pDC's), we depleted these cells prior to LDV infection and tested for lymphocyte activation. Depletion of pDC's in vivo eradicated both the LDV-induced IFNalpha response and lymphocyte activation. A primary receptor in pDC's for single stranded RNA viruses such as LDV is the toll-like receptor 7 (TLR7) pattern recognition receptor. Infection of TLR7-knockout mice revealed that both the IFNalpha response and lymphocyte activation were dependent on TLR7 signaling in vivo. Interestingly, virus levels in both TLR7 knockout mice and pDC-depleted mice were indistinguishable from controls indicating that LDV is largely resistant to the systemic IFNalpha response. CONCLUSION: Results indicate that LDV-induced activation of lymphocytes is due to recognition of LDV nucleic acid by TLR7 pattern recognition receptors in pDC's that respond with a lymphocyte-inducing IFNalpha response.\", 'denotations': [{'obj': 'Disease:D015658', 'span': {'begin': 610, 'end': 623}}, {'obj': 'Disease:D015658', 'span': {'begin': 878, 'end': 891}}, {'obj': 'Chemical:D019344', 'span': {'begin': 160, 'end': 167}}, {'obj': 'Chemical:D019344', 'span': {'begin': 0, 'end': 7}}, {'obj': 'Gene:51284', 'span': {'begin': 81, 'end': 85}}, {'obj': 'Gene:3439', 'span': {'begin': 96, 'end': 104}}, {'obj': 'Gene:111654', 'span': {'begin': 569, 'end': 577}}, {'obj': 'Gene:111654', 'span': {'begin': 643, 'end': 651}}, {'obj': 'Gene:111654', 'span': {'begin': 781, 'end': 789}}, {'obj': 'Gene:111654', 'span': {'begin': 989, 'end': 997}}, {'obj': 'Gene:170743', 'span': {'begin': 1135, 'end': 1139}}, {'obj': 'Gene:170743', 'span': {'begin': 1184, 'end': 1188}}, {'obj': 'Gene:111654', 'span': {'begin': 1226, 'end': 1234}}, {'obj': 'Gene:170743', 'span': {'begin': 1288, 'end': 1292}}, {'obj': 'Gene:170743', 'span': {'begin': 1348, 'end': 1352}}, {'obj': 'Gene:111654', 'span': {'begin': 1483, 'end': 1491}}, {'obj': 'Gene:170743', 'span': {'begin': 1619, 'end': 1623}}, {'obj': 'Gene:111654', 'span': {'begin': 1703, 'end': 1711}}, {'obj': 'Species:10090', 'span': {'begin': 237, 'end': 241}}, {'obj': 'Species:10239', 'span': {'begin': 262, 'end': 269}}, {'obj': 'Species:11048', 'span': {'begin': 271, 'end': 274}}, {'obj': 'Species:10090', 'span': {'begin': 680, 'end': 684}}, {'obj': 'Species:12081', 'span': {'begin': 767, 'end': 772}}, {'obj': 'Species:11048', 'span': {'begin': 977, 'end': 980}}, {'obj': 'Species:10239', 'span': {'begin': 1086, 'end': 1093}}, {'obj': 'Species:11048', 'span': {'begin': 1102, 'end': 1105}}, {'obj': 'Species:10090', 'span': {'begin': 1198, 'end': 1202}}, {'obj': 'Species:12081', 'span': {'begin': 1327, 'end': 1332}}, {'obj': 'Species:10090', 'span': {'begin': 1362, 'end': 1366}}, {'obj': 'Species:10090', 'span': {'begin': 1384, 'end': 1388}}, {'obj': 'Species:11048', 'span': {'begin': 1442, 'end': 1445}}, {'obj': 'Species:11048', 'span': {'begin': 1536, 'end': 1539}}, {'obj': 'Species:11048', 'span': {'begin': 1599, 'end': 1602}}]}]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'sourceid': '19635124', 'sourcedb': 'PubMed', 'text': 'RNA viruses in community-acquired childhood pneumonia in semi-urban Nepal; a cross-sectional study. BACKGROUND: Pneumonia is among the main causes of illness and death in children <5 years of age. There is a need to better describe the epidemiology of viral community-acquired pneumonia (CAP) in developing countries. METHODS: From July 2004 to June 2007, we examined nasopharyngeal aspirates (NPA) from 2,230 cases of pneumonia (World Health Organization criteria) in children 2 to 35 months old recruited in a randomized trial of zinc supplementation at a field clinic in Bhaktapur, Nepal. The specimens were examined for respiratory syncytial virus (RSV), influenza virus type A (InfA) and B (InfB), parainfluenza virus types 1, 2 and 3 (PIV1, PIV2, and PIV3), and human metapneumovirus (hMPV) using a multiplex reverse transcriptase polymerase chain reaction (PCR) assay. RESULTS: We identified 919 virus isolates in 887 (40.0%) of the 2,219 NPA specimens with a valid PCR result, of which 334 (15.1%) yielded RSV, 164 (7.4%) InfA, 129 (5.8%) PIV3, 98 (4.4%) PIV1, 93 (4.2%) hMPV, 84 (3.8%) InfB, and 17 (0.8%) PIV2. CAP occurred in an epidemic pattern with substantial temporal variation during the three years of study. The largest peaks of pneumonia occurrence coincided with peaks of RSV infection, which occurred in epidemics during the rainy season and in winter. The monthly number of RSV infections was positively correlated with relative humidity (rs = 0.40, P = 0.01), but not with temperature or rainfall. An hMPV epidemic occurred during one of the three winter seasons and the monthly number of hMPV cases was also associated with relative humidity (rs = 0.55, P = 0.0005). CONCLUSION: Respiratory RNA viruses were detected from NPA in 40% of CAP cases in our study. The most commonly isolated viruses were RSV, InfA, and PIV3. RSV infections contributed substantially to the observed CAP epidemics. The occurrence of viral CAP in this community seemed to reflect more or less overlapping micro-epidemics with several respiratory viruses, highlighting the challenges of developing and implementing effective public health control measures.', 'denotations': [{'obj': 'Disease:D011014', 'span': {'begin': 34, 'end': 53}}, {'obj': 'Disease:D003643', 'span': {'begin': 162, 'end': 167}}, {'obj': 'Disease:D003147', 'span': {'begin': 258, 'end': 286}}, {'obj': 'Disease:D003147', 'span': {'begin': 288, 'end': 291}}, {'obj': 'Disease:D011014', 'span': {'begin': 419, 'end': 428}}, {'obj': 'Disease:D018357', 'span': {'begin': 624, 'end': 681}}, {'obj': 'Disease:D018184', 'span': {'begin': 703, 'end': 716}}, {'obj': 'Disease:D003147', 'span': {'begin': 1121, 'end': 1124}}, {'obj': 'Disease:D004671', 'span': {'begin': 1140, 'end': 1148}}, {'obj': 'Disease:D011014', 'span': {'begin': 1247, 'end': 1256}}, {'obj': 'Disease:D018357', 'span': {'begin': 1292, 'end': 1305}}, {'obj': 'Disease:D018357', 'span': {'begin': 1396, 'end': 1410}}, {'obj': 'Disease:D004671', 'span': {'begin': 1529, 'end': 1537}}, {'obj': 'Disease:D003147', 'span': {'begin': 1760, 'end': 1763}}, {'obj': 'Disease:D018357', 'span': {'begin': 1845, 'end': 1859}}, {'obj': 'Disease:D003147', 'span': {'begin': 1902, 'end': 1905}}, {'obj': 'Disease:D011024', 'span': {'begin': 1935, 'end': 1944}}, {'obj': 'Chemical:CHEBI:60833', 'span': {'begin': 1746, 'end': 1749}}, {'obj': 'Chemical:D015032', 'span': {'begin': 532, 'end': 536}}, {'obj': 'Species:9606', 'span': {'begin': 171, 'end': 179}}, {'obj': 'Species:9606', 'span': {'begin': 469, 'end': 477}}, {'obj': 'Species:11212', 'span': {'begin': 747, 'end': 751}}, {'obj': 'Species:162145', 'span': {'begin': 768, 'end': 789}}, {'obj': 'Species:162145', 'span': {'begin': 791, 'end': 795}}, {'obj': 'Species:11886', 'span': {'begin': 1014, 'end': 1017}}, {'obj': 'Species:162145', 'span': {'begin': 1079, 'end': 1083}}, {'obj': 'Species:11212', 'span': {'begin': 1115, 'end': 1119}}, {'obj': 'Species:162145', 'span': {'begin': 1524, 'end': 1528}}, {'obj': 'Species:162145', 'span': {'begin': 1612, 'end': 1616}}, {'obj': 'Species:12814', 'span': {'begin': 1703, 'end': 1714}}, {'obj': 'Species:11886', 'span': {'begin': 1824, 'end': 1827}}, {'obj': 'Species:12814', 'span': {'begin': 2035, 'end': 2046}}]}]\n",
      "[{'sourceid': '19682364', 'sourcedb': 'PubMed', 'text': 'SNAD: Sequence Name Annotation-based Designer. BACKGROUND: A growing diversity of biological data is tagged with unique identifiers (UIDs) associated with polynucleotides and proteins to ensure efficient computer-mediated data storage, maintenance, and processing. These identifiers, which are not informative for most people, are often substituted by biologically meaningful names in various presentations to facilitate utilization and dissemination of sequence-based knowledge. This substitution is commonly done manually that may be a tedious exercise prone to mistakes and omissions. RESULTS: Here we introduce SNAD (Sequence Name Annotation-based Designer) that mediates automatic conversion of sequence UIDs (associated with multiple alignment or phylogenetic tree, or supplied as plain text list) into biologically meaningful names and acronyms. This conversion is directed by precompiled or user-defined templates that exploit wealth of annotation available in cognate entries of external databases. Using examples, we demonstrate how this tool can be used to generate names for practical purposes, particularly in virology. CONCLUSION: A tool for controllable annotation-based conversion of sequence UIDs into biologically meaningful names and acronyms has been developed and placed into service, fostering links between quality of sequence annotation, and efficiency of communication and knowledge dissemination among researchers.', 'denotations': [{'obj': 'Disease:C566733', 'span': {'begin': 113, 'end': 131}}, {'obj': 'Disease:C566733', 'span': {'begin': 133, 'end': 137}}, {'obj': 'Disease:C566733', 'span': {'begin': 709, 'end': 713}}, {'obj': 'Disease:C566733', 'span': {'begin': 1209, 'end': 1213}}, {'obj': 'Species:9606', 'span': {'begin': 319, 'end': 325}}]}]\n",
      "[{'sourceid': '20015400', 'sourcedb': 'PubMed', 'text': 'Public perceptions of quarantine: community-based telephone survey following an infectious disease outbreak. BACKGROUND: The use of restrictive measures such as quarantine draws into sharp relief the dynamic interplay between the individual rights of the citizen on the one hand and the collective rights of the community on the other. Concerns regarding infectious disease outbreaks (SARS, pandemic influenza) have intensified the need to understand public perceptions of quarantine and other social distancing measures. METHODS: We conducted a telephone survey of the general population in the Greater Toronto Area in Ontario, Canada. Computer-assisted telephone interviewing (CATI) technology was used. A final sample of 500 individuals was achieved through standard random-digit dialing. RESULTS: Our data indicate strong public support for the use of quarantine when required and for serious legal sanctions against those who fail to comply. This support is contingent both on the implementation of legal safeguards to protect against inappropriate use and on the provision of psychosocial supports for those affected. CONCLUSION: To engender strong public support for quarantine and other restrictive measures, government officials and public health policy-makers would do well to implement a comprehensive system of supports and safeguards, to educate and inform frontline public health workers, and to engage the public at large in an open dialogue on the ethical use of restrictive measures during infectious disease outbreaks.', 'denotations': [{'obj': 'Disease:D003141', 'span': {'begin': 80, 'end': 98}}, {'obj': 'Disease:D003141', 'span': {'begin': 355, 'end': 373}}, {'obj': 'Disease:D045169', 'span': {'begin': 385, 'end': 389}}, {'obj': 'Disease:D007251', 'span': {'begin': 400, 'end': 409}}, {'obj': 'Disease:D003141', 'span': {'begin': 1507, 'end': 1525}}, {'obj': 'Chemical:', 'span': {'begin': 1174, 'end': 1184}}, {'obj': 'Chemical:', 'span': {'begin': 856, 'end': 866}}, {'obj': 'Chemical:', 'span': {'begin': 473, 'end': 483}}, {'obj': 'Chemical:', 'span': {'begin': 161, 'end': 171}}, {'obj': 'Chemical:', 'span': {'begin': 22, 'end': 32}}]}]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'sourceid': '21408132', 'sourcedb': 'PubMed', 'text': 'Risk factors of Streptococcus suis infection in Vietnam. A case-control study. BACKGROUND: Streptococcus suis infection, an emerging zoonosis, is an increasing public health problem across South East Asia and the most common cause of acute bacterial meningitis in adults in Vietnam. Little is known of the risk factors underlying the disease. METHODS AND FINDINGS: A case-control study with appropriate hospital and matched community controls for each patient was conducted between May 2006 and June 2009. Potential risk factors were assessed using a standardized questionnaire and investigation of throat and rectal S. suis carriage in cases, controls and their pigs, using real-time PCR and culture of swab samples. We recruited 101 cases of S. suis meningitis, 303 hospital controls and 300 community controls. By multivariate analysis, risk factors identified for S. suis infection as compared to either control group included eating \"high risk\" dishes, including such dishes as undercooked pig blood and pig intestine (OR(1)=2.22; 95%CI=[1.15-4.28] and OR(2)=4.44; 95%CI=[2.15-9.15]), occupations related to pigs (OR(1)=3.84; 95%CI=[1.32-11.11] and OR(2)=5.52; 95%CI=[1.49-20.39]), and exposures to pigs or pork in the presence of skin injuries (OR(1)=7.48; 95%CI=[1.97-28.44] and OR(2)=15.96; 95%CI=[2.97-85.72]). S. suis specific DNA was detected in rectal and throat swabs of 6 patients and was cultured from 2 rectal samples, but was not detected in such samples of 1522 healthy individuals or patients without S. suis infection. CONCLUSIONS: This case control study, the largest prospective epidemiological assessment of this disease, has identified the most important risk factors associated with S. suis bacterial meningitis to be eating \\'high risk\\' dishes popular in parts of Asia, occupational exposure to pigs and pig products, and preparation of pork in the presence of skin lesions. These risk factors can be addressed in public health campaigns aimed at preventing S. suis infection.', 'denotations': [{'obj': 'Disease:D007239', 'span': {'begin': 91, 'end': 119}}, {'obj': 'Disease:D007239', 'span': {'begin': 16, 'end': 44}}, {'obj': 'Disease:D016920', 'span': {'begin': 240, 'end': 260}}, {'obj': 'Disease:D008581', 'span': {'begin': 752, 'end': 762}}, {'obj': 'Disease:D012871', 'span': {'begin': 1236, 'end': 1249}}, {'obj': 'Disease:D016920', 'span': {'begin': 1716, 'end': 1736}}, {'obj': 'Disease:D012871', 'span': {'begin': 1886, 'end': 1898}}, {'obj': 'Species:9606', 'span': {'begin': 452, 'end': 459}}, {'obj': 'Species:1307', 'span': {'begin': 617, 'end': 624}}, {'obj': 'Species:9823', 'span': {'begin': 663, 'end': 667}}, {'obj': 'Species:1307', 'span': {'begin': 744, 'end': 751}}, {'obj': 'Species:1307', 'span': {'begin': 868, 'end': 875}}, {'obj': 'Species:9823', 'span': {'begin': 995, 'end': 998}}, {'obj': 'Species:9823', 'span': {'begin': 1009, 'end': 1012}}, {'obj': 'Species:9823', 'span': {'begin': 1113, 'end': 1117}}, {'obj': 'Species:9823', 'span': {'begin': 1204, 'end': 1208}}, {'obj': 'Species:1307', 'span': {'begin': 1320, 'end': 1327}}, {'obj': 'Species:9606', 'span': {'begin': 1386, 'end': 1394}}, {'obj': 'Species:9606', 'span': {'begin': 1503, 'end': 1511}}, {'obj': 'Species:1307', 'span': {'begin': 1520, 'end': 1527}}, {'obj': 'Species:1307', 'span': {'begin': 1708, 'end': 1715}}, {'obj': 'Species:9823', 'span': {'begin': 1820, 'end': 1824}}, {'obj': 'Species:9823', 'span': {'begin': 1829, 'end': 1832}}, {'obj': 'Species:1307', 'span': {'begin': 1983, 'end': 1990}}]}]\n",
      "[{'sourceid': '21450075', 'sourcedb': 'PubMed', 'text': 'Analysis of codon usage and nucleotide composition bias in polioviruses. BACKGROUND: Poliovirus, the causative agent of poliomyelitis, is a human enterovirus and a member of the family of Picornaviridae and among the most rapidly evolving viruses known. Analysis of codon usage can reveal much about the molecular evolution of the viruses. However, little information about synonymous codon usage pattern of polioviruses genome has been acquired to date. METHODS: The relative synonymous codon usage (RSCU) values, effective number of codon (ENC) values, nucleotide contents and dinucleotides were investigated and a comparative analysis of codon usage pattern for open reading frames (ORFs) among 48 polioviruses isolates including 31 of genotype 1, 13 of genotype 2 and 4 of genotype 3. RESULTS: The result shows that the overall extent of codon usage bias in poliovirus samples is low (mean ENC = 53.754 > 40). The general correlation between base composition and codon usage bias suggests that mutational pressure rather than natural selection is the main factor that determines the codon usage bias in those polioviruses. Depending on the RSCU data, it was found that there was a significant variation in bias of codon usage among three genotypes. Geographic factor also has some effect on the codon usage pattern (exists in the genotype-1 of polioviruses). No significant effect in gene length or vaccine derived polioviruses (DVPVs), wild viruses and live attenuated virus was observed on the variations of synonymous codon usage in the virus genes. The relative abundance of dinucleotide (CpG) in the ORFs of polioviruses are far below expected values especially in DVPVs and attenuated virus of polioviruses genotype 1. CONCLUSION: The information from this study may not only have theoretical value in understanding poliovirus evolution, especially for DVPVs genotype 1, but also have potential value for the development of poliovirus vaccines.', 'denotations': [{'obj': 'Disease:D011051', 'span': {'begin': 120, 'end': 133}}, {'obj': 'Chemical:', 'span': {'begin': 1597, 'end': 1600}}, {'obj': 'Chemical:CHEBI:47885', 'span': {'begin': 1583, 'end': 1595}}, {'obj': 'Chemical:CHEBI:47885', 'span': {'begin': 579, 'end': 592}}, {'obj': 'Chemical:CHEBI:36976', 'span': {'begin': 555, 'end': 565}}, {'obj': 'Chemical:CHEBI:36976', 'span': {'begin': 28, 'end': 38}}, {'obj': 'Species:138950', 'span': {'begin': 85, 'end': 95}}, {'obj': 'Species:9606', 'span': {'begin': 140, 'end': 145}}, {'obj': 'Species:138950', 'span': {'begin': 862, 'end': 872}}, {'obj': 'Species:138950', 'span': {'begin': 1826, 'end': 1836}}, {'obj': 'Species:138950', 'span': {'begin': 1934, 'end': 1944}}]}]\n",
      "[{'sourceid': '22726254', 'sourcedb': 'PubMed', 'text': 'In young children, persistent wheezing is associated with bronchial bacterial infection: a retrospective analysis. BACKGROUND: Young children with persistent wheezing pose a diagnostic and therapeutical challenge to the pediatrician.We aimed to evaluate bacterial bronchial infection as a possible reason for non response to conventional asthma therapy, and to identify and characterise the predominant pathogens involved. METHODS: We retrospectively analysed microbiological and cytological findings in a selected population of young wheezers with symptoms unresponsive to inhaled corticosteroid (ICS) therapy, who underwent flexible bronchoscopy with bronchoalveolar lavage (BAL). Procedural measures were taken to limit contamination risk and quantitative bacterial culture of BAL fluid (significance cut-off >=  10    colony-forming units/ml) was used. Modern microbiological methods were used for detection of a wide panel of pathogens and for characterisation of the bacterial isolates. RESULTS: 33 children aged between 4 and 38 months, without structural anomalies of the conductive airways were evaluated. Significant bacterial BAL cultures were found in 48,5 % of patients. Haemophilus influenzae was isolated in 30,3 %, Streptococcus pneumoniae in 12,1 % and Moraxella catarrhalis in 12,1 %. All H. influenzae isolates were non-encapsulated strains and definitely distinguished from non-haemolytic H. haemolyticus. Respiratory viruses were detected in 21,9 % of cases with mixed bacterial-viral infection in 12,1 %. Cytology revealed a marked neutrophilic inflammation. CONCLUSIONS: Bacterial infection of the bronchial tree is common in persistent preschool wheezers and provides a possible explanation for non response to ICS therapy. Non-typeable H. influenzae seems to be the predominant pathogen involved, followed by S. pneumoniae and M. catarrhalis.', 'denotations': [{'obj': 'Disease:D001424', 'span': {'begin': 58, 'end': 87}}, {'obj': 'Disease:D001424', 'span': {'begin': 254, 'end': 283}}, {'obj': 'Disease:D001424', 'span': {'begin': 1127, 'end': 1149}}, {'obj': 'Disease:D001424', 'span': {'begin': 1490, 'end': 1515}}, {'obj': 'Disease:D007249', 'span': {'begin': 1554, 'end': 1579}}, {'obj': 'Disease:D001424', 'span': {'begin': 1594, 'end': 1613}}, {'obj': 'Species:9606', 'span': {'begin': 9, 'end': 17}}, {'obj': 'Species:9606', 'span': {'begin': 133, 'end': 141}}, {'obj': 'Species:9606', 'span': {'begin': 1005, 'end': 1013}}, {'obj': 'Species:9606', 'span': {'begin': 1174, 'end': 1182}}, {'obj': 'Species:727', 'span': {'begin': 1184, 'end': 1206}}, {'obj': 'Species:1313', 'span': {'begin': 1231, 'end': 1255}}, {'obj': 'Species:480', 'span': {'begin': 1270, 'end': 1291}}, {'obj': 'Species:727', 'span': {'begin': 1307, 'end': 1320}}, {'obj': 'Species:727', 'span': {'begin': 1761, 'end': 1774}}, {'obj': 'Species:2104', 'span': {'begin': 1834, 'end': 1847}}, {'obj': 'Species:480', 'span': {'begin': 1852, 'end': 1866}}]}]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'sourceid': '23028498', 'sourcedb': 'PubMed', 'text': 'Temporal percolation of the susceptible network in an epidemic spreading. In this work, we study the evolution of the susceptible individuals during the spread of an epidemic modeled by the susceptible-infected-recovered (SIR) process spreading on the top of complex networks. Using an edge-based compartmental approach and percolation tools, we find that a time-dependent quantity   S(t), namely, the probability that a given neighbor of a node is susceptible at time t, is the control parameter of a node void percolation process involving those nodes on the network not-reached by the disease. We show that there exists a critical time t(c) above which the giant susceptible component is destroyed. As a consequence, in order to preserve a macroscopic connected fraction of the network composed by healthy individuals which guarantee its functionality, any mitigation strategy should be implemented before this critical time t(c). Our theoretical results are confirmed by extensive simulations of the SIR process.', 'denotations': [{'obj': 'Disease:D004671', 'span': {'begin': 54, 'end': 62}}, {'obj': 'Disease:D004671', 'span': {'begin': 166, 'end': 174}}]}]\n",
      "[{'sourceid': '23074659', 'sourcedb': 'PubMed', 'text': 'Blow Flies Were One of the Possible Candidates for Transmission of Highly Pathogenic H5N1 Avian Influenza Virus during the 2004 Outbreaks in Japan. The 2003-2004 H5N1 highly pathogenic avian influenza (HPAI) outbreaks in Japan were the first such outbreaks in 79 years in Japan. Epidemic outbreaks have been occurring in Southeast Asia, with the most recent in 2010. Knowledge of the transmission route responsible for the HPAI outbreaks in these countries remains elusive. Our studies strongly suggested that field and laboratory studies focusing on mechanical transmission by blow flies should be considered to control H5N1 avian influenza outbreaks, in particular in epidemic areas, where there are high densities of different fly species throughout the year. In this paper, we review these field and laboratory entomological studies and discuss the possibility of blow flies transmitting H5N1 viruses.', 'denotations': [{'obj': 'Disease:D009917', 'span': {'begin': 578, 'end': 582}}, {'obj': 'Disease:D009917', 'span': {'begin': 868, 'end': 872}}, {'obj': 'Species:7227', 'span': {'begin': 730, 'end': 733}}]}]\n",
      "[{'sourceid': '23110173', 'sourcedb': 'PubMed', 'text': 'SEED servers: high-performance access to the SEED genomes, annotations, and metabolic models. The remarkable advance in sequencing technology and the rising interest in medical and environmental microbiology, biotechnology, and synthetic biology resulted in a deluge of published microbial genomes. Yet, genome annotation, comparison, and modeling remain a major bottleneck to the translation of sequence information into biological knowledge, hence computational analysis tools are continuously being developed for rapid genome annotation and interpretation. Among the earliest, most comprehensive resources for prokaryotic genome analysis, the SEED project, initiated in 2003 as an integration of genomic data and analysis tools, now contains >5,000 complete genomes, a constantly updated set of curated annotations embodied in a large and growing collection of encoded subsystems, a derived set of protein families, and hundreds of genome-scale metabolic models. Until recently, however, maintaining current copies of the SEED code and data at remote locations has been a pressing issue. To allow high-performance remote access to the SEED database, we developed the SEED Servers (http://www.theseed.org/servers): four network-based servers intended to expose the data in the underlying relational database, support basic annotation services, offer programmatic access to the capabilities of the RAST annotation server, and provide access to a growing collection of metabolic models that support flux balance analysis. The SEED servers offer open access to regularly updated data, the ability to annotate prokaryotic genomes, the ability to create metabolic reconstructions and detailed models of metabolism, and access to hundreds of existing metabolic models. This work offers and supports a framework upon which other groups can build independent research efforts. Large integrations of genomic data represent one of the major intellectual resources driving research in biology, and programmatic access to the SEED data will provide significant utility to a broad collection of potential users.', 'denotations': []}]\n",
      "[{'sourceid': '23193411', 'sourcedb': 'PubMed', 'text': 'Megapneumonia Coinfection: pneumococcus, Mycoplasma pneumoniae, and Metapneumovirus. We report a young girl who died of Streptococcus pneumoniae 19A pneumonia, septic shock, and hemolytic uremic syndrome despite prior pneumococcal vaccination, appropriate antibiotics, and aggressive intensive care support. Serotype 19A is not covered by the 7- or 10-valent pneumococcal vaccines. Mycoplasma pneumoniae and metapneumovirus were simultaneously detected by PCR in the nasopharyngeal and tracheal aspirates. The pneumococcus is penicillin sensitive. Although infections with each of these pathogens alone are typically mild, this case highlights that co-infection with the triple respiratory pathogens possibly contributed to the fatal outcome of this child. Also, the new policy in Hong Kong to use PCV13 may help prevent further cases of serotype 19A infections.', 'denotations': [{'obj': 'Disease:D011019', 'span': {'begin': 41, 'end': 62}}, {'obj': 'Disease:D011014', 'span': {'begin': 149, 'end': 158}}, {'obj': 'Disease:D012772', 'span': {'begin': 160, 'end': 172}}, {'obj': 'Disease:D006463', 'span': {'begin': 178, 'end': 203}}, {'obj': 'Disease:D001523', 'span': {'begin': 273, 'end': 283}}, {'obj': 'Disease:D011019', 'span': {'begin': 382, 'end': 403}}, {'obj': 'Disease:D007239', 'span': {'begin': 557, 'end': 567}}, {'obj': 'Disease:D060085', 'span': {'begin': 649, 'end': 661}}, {'obj': 'Disease:D007239', 'span': {'begin': 851, 'end': 861}}, {'obj': 'Chemical:D010406', 'span': {'begin': 526, 'end': 536}}, {'obj': 'Species:9606', 'span': {'begin': 103, 'end': 107}}, {'obj': 'Species:1313', 'span': {'begin': 120, 'end': 144}}, {'obj': 'Species:9606', 'span': {'begin': 750, 'end': 755}}]}]\n",
      "[{'sourceid': '22590978', 'sourcedb': 'PubMed', 'text': 'The ORF2 glycoprotein of hepatitis E virus inhibits cellular NF-kB activity by blocking ubiquitination mediated proteasomal degradation of IkBa in human hepatoma cells. BACKGROUND: Nuclear factor kappa B (NF-kB) is a key transcription factor that plays a crucial role in host survival during infection by pathogens. Therefore, it has been a priority of many pathogens to manipulate the cellular NF-kB activity in order to create a favorable environment for their survival inside the host. RESULTS: We observed that heterologous expression of the open reading frame 2 (ORF2) protein in human hepatoma cells led to stabilization of the cellular I kappa B alpha (IkBa) pool, with a concomitant reduction in the nuclear localization of the p65 subunit of NF-kB and inhibition of NF-kB activity. Although basal or TPA induced phosphorylation of IkBa was not altered, its ubiquitination was markedly reduced in ORF2 expressing cells. Further analysis revealed that ORF2 protein could directly associate with the F-box protein, beta transducin repeat containing protein (bTRCP) and ORF2 over expression resulted in reduced association of IkBa with the SKP1 and CUL1 components of the SCFbTRCP complex. Chromatin immunoprecipitation (ChIP) assay of the proximal promoter regions of MHC-I heavy chain and IL-8 genes using p65 antibody and LPS stimulated ORF2 expressing cell extract revealed decreased association of p65 with the above regions, indicating that ORF2 inhibited p65 binding at endogenous promoters. CONCLUSIONS: In this report we suggest a mechanism by which ORF2 protein of HEV may inhibit host cell NF-kB activity during the course of a viral infection.', 'denotations': [{'obj': 'Disease:D056486', 'span': {'begin': 25, 'end': 36}}, {'obj': 'Disease:D006528', 'span': {'begin': 153, 'end': 161}}, {'obj': 'Disease:D006528', 'span': {'begin': 591, 'end': 599}}, {'obj': 'Disease:C536528', 'span': {'begin': 1330, 'end': 1333}}, {'obj': 'Disease:D014777', 'span': {'begin': 1644, 'end': 1659}}, {'obj': 'Gene:4792', 'span': {'begin': 139, 'end': 143}}, {'obj': 'Gene:4792', 'span': {'begin': 643, 'end': 658}}, {'obj': 'Gene:4792', 'span': {'begin': 660, 'end': 664}}, {'obj': 'Gene:5970', 'span': {'begin': 736, 'end': 739}}, {'obj': 'Gene:4792', 'span': {'begin': 840, 'end': 844}}, {'obj': 'Gene:8945', 'span': {'begin': 1021, 'end': 1062}}, {'obj': 'Gene:8945', 'span': {'begin': 1064, 'end': 1069}}, {'obj': 'Gene:4792', 'span': {'begin': 1131, 'end': 1135}}, {'obj': 'Gene:6500', 'span': {'begin': 1145, 'end': 1149}}, {'obj': 'Gene:8454', 'span': {'begin': 1154, 'end': 1158}}, {'obj': 'Gene:3576', 'span': {'begin': 1296, 'end': 1300}}, {'obj': 'Gene:5970', 'span': {'begin': 1313, 'end': 1316}}, {'obj': 'Gene:5970', 'span': {'begin': 1408, 'end': 1411}}, {'obj': 'Gene:5970', 'span': {'begin': 1467, 'end': 1470}}, {'obj': 'Species:9606', 'span': {'begin': 147, 'end': 152}}, {'obj': 'Species:9606', 'span': {'begin': 585, 'end': 590}}, {'obj': 'Species:63330', 'span': {'begin': 1580, 'end': 1583}}]}]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'sourceid': '23062055', 'sourcedb': 'PubMed', 'text': 'A new look at an old virus: patterns of mutation accumulation in the human H1N1 influenza virus since 1918. BACKGROUND: The H1N1 influenza A virus has been circulating in the human population for over 95 years, first manifesting itself in the pandemic of 1917-1918. Initial mortality was extremely high, but dropped exponentially over time. Influenza viruses have high mutation rates, and H1N1 has undergone significant genetic changes since 1918. The exact nature of H1N1 mutation accumulation over time has not been fully explored. METHODS: We have made a comprehensive historical analysis of mutational changes within H1N1 by examining over 4100 fully-sequenced H1N1 genomes. This has allowed us to examine the genetic changes arising within H1N1 from 1918 to the present. RESULTS: We document multiple extinction events, including the previously known extinction of the human H1N1 lineage in the 1950s, and an apparent second extinction of the human H1N1 lineage in 2009. These extinctions appear to be due to a continuous accumulation of mutations. At the time of its disappearance in 2009, the human H1N1 lineage had accumulated over 1400 point mutations (more than 10% of the genome), including approximately 330 non-synonymous changes (7.4% of all codons). The accumulation of both point mutations and non-synonymous amino acid changes occurred at constant rates (  = 14.4 and 2.4 new mutations/year, respectively), and mutations accumulated uniformly across the entire influenza genome. We observed a continuous erosion over time of codon-specificity in H1N1, including a shift away from host (human, swine, and bird [duck]) codon preference patterns. CONCLUSIONS: While there have been numerous adaptations within the H1N1 genome, most of the genetic changes we document here appear to be non-adaptive, and much of the change appears to be degenerative. We suggest H1N1 has been undergoing natural genetic attenuation, and that significant attenuation may even occur during a single pandemic. This process may play a role in natural pandemic cessation and has apparently contributed to the exponential decline in mortality rates over time, as seen in all major human influenza strains. These findings may be relevant to the development of strategies for managing influenza pandemics and strain evolution.', 'denotations': [{'obj': 'Disease:D007251', 'span': {'begin': 129, 'end': 138}}, {'obj': 'Disease:D007251', 'span': {'begin': 80, 'end': 89}}, {'obj': 'Disease:D007251', 'span': {'begin': 1478, 'end': 1487}}, {'obj': 'Disease:D007251', 'span': {'begin': 2177, 'end': 2186}}, {'obj': 'Disease:D007251', 'span': {'begin': 2273, 'end': 2282}}, {'obj': 'Chemical:CHEBI:33704', 'span': {'begin': 1325, 'end': 1335}}, {'obj': 'Species:9606', 'span': {'begin': 175, 'end': 180}}, {'obj': 'Species:9606', 'span': {'begin': 69, 'end': 74}}, {'obj': 'Species:9606', 'span': {'begin': 874, 'end': 879}}, {'obj': 'Species:9606', 'span': {'begin': 948, 'end': 953}}, {'obj': 'Species:9606', 'span': {'begin': 1100, 'end': 1105}}, {'obj': 'Species:9606', 'span': {'begin': 1603, 'end': 1608}}, {'obj': 'Species:9823', 'span': {'begin': 1610, 'end': 1615}}, {'obj': 'Species:8839', 'span': {'begin': 1627, 'end': 1631}}, {'obj': 'Species:9606', 'span': {'begin': 2171, 'end': 2176}}]}]\n",
      "[{'sourceid': '23284733', 'sourcedb': 'PubMed', 'text': 'Immunization with a recombinant vaccinia virus that encodes nonstructural proteins of the hepatitis C virus suppresses viral protein levels in mouse liver. Chronic hepatitis C, which is caused by infection with the hepatitis C virus (HCV), is a global health problem. Using a mouse model of hepatitis C, we examined the therapeutic effects of a recombinant vaccinia virus (rVV) that encodes an HCV protein. We generated immunocompetent mice that each expressed multiple HCV proteins via a Cre/loxP switching system and established several distinct attenuated rVV strains. The HCV core protein was expressed consistently in the liver after polyinosinic acid-polycytidylic acid injection, and these mice showed chronic hepatitis C-related pathological findings (hepatocyte abnormalities, accumulation of glycogen, steatosis), liver fibrosis, and hepatocellular carcinoma. Immunization with one rVV strain (rVV-N25), which encoded nonstructural HCV proteins, suppressed serum inflammatory cytokine levels and alleviated the symptoms of pathological chronic hepatitis C within 7 days after injection. Furthermore, HCV protein levels in liver tissue also decreased in a CD4 and CD8 T-cell-dependent manner. Consistent with these results, we showed that rVV-N25 immunization induced a robust CD8 T-cell immune response that was specific to the HCV nonstructural protein 2. We also demonstrated that the onset of chronic hepatitis in CN2-29((+/-))/MxCre((+/-)) mice was mainly attributable to inflammatory cytokines, (tumor necrosis factor) TNF-a and (interleukin) IL-6. Thus, our generated mice model should be useful for further investigation of the immunological processes associated with persistent expression of HCV proteins because these mice had not developed immune tolerance to the HCV antigen. In addition, we propose that rVV-N25 could be developed as an effective therapeutic vaccine.', 'denotations': [{'obj': 'Disease:D019698', 'span': {'begin': 156, 'end': 175}}, {'obj': 'Disease:D006526', 'span': {'begin': 196, 'end': 232}}, {'obj': 'Disease:D006526', 'span': {'begin': 74, 'end': 101}}, {'obj': 'Disease:D006526', 'span': {'begin': 291, 'end': 302}}, {'obj': 'Disease:D019698', 'span': {'begin': 709, 'end': 728}}, {'obj': 'Disease:D005234', 'span': {'begin': 812, 'end': 821}}, {'obj': 'Disease:D005355', 'span': {'begin': 830, 'end': 838}}, {'obj': 'Disease:D006528', 'span': {'begin': 844, 'end': 868}}, {'obj': 'Disease:D019698', 'span': {'begin': 1033, 'end': 1065}}, {'obj': 'Disease:D006526', 'span': {'begin': 1338, 'end': 1365}}, {'obj': 'Disease:D006521', 'span': {'begin': 1406, 'end': 1423}}, {'obj': 'Disease:D009369', 'span': {'begin': 1511, 'end': 1516}}, {'obj': 'Chemical:', 'span': {'begin': 657, 'end': 675}}, {'obj': 'Chemical:', 'span': {'begin': 639, 'end': 656}}, {'obj': 'Gene:21926', 'span': {'begin': 1534, 'end': 1562}}, {'obj': 'Species:10245', 'span': {'begin': 32, 'end': 46}}, {'obj': 'Species:10090', 'span': {'begin': 276, 'end': 281}}, {'obj': 'Species:10090', 'span': {'begin': 143, 'end': 148}}, {'obj': 'Species:10245', 'span': {'begin': 357, 'end': 371}}, {'obj': 'Species:11096', 'span': {'begin': 394, 'end': 397}}, {'obj': 'Species:10090', 'span': {'begin': 436, 'end': 440}}, {'obj': 'Species:11096', 'span': {'begin': 470, 'end': 473}}, {'obj': 'Species:11096', 'span': {'begin': 576, 'end': 579}}, {'obj': 'Species:10090', 'span': {'begin': 697, 'end': 701}}, {'obj': 'Species:11096', 'span': {'begin': 942, 'end': 945}}, {'obj': 'Species:11096', 'span': {'begin': 1110, 'end': 1113}}, {'obj': 'Species:10090', 'span': {'begin': 1454, 'end': 1458}}, {'obj': 'Species:10090', 'span': {'begin': 1584, 'end': 1588}}, {'obj': 'Species:11096', 'span': {'begin': 1710, 'end': 1713}}, {'obj': 'Species:10090', 'span': {'begin': 1737, 'end': 1741}}]}]\n",
      "[{'sourceid': '23202545', 'sourcedb': 'PubMed', 'text': 'Involvement of autophagy in coronavirus replication. Coronaviruses are single stranded, positive sense RNA viruses, which induce the rearrangement of cellular membranes upon infection of a host cell. This provides the virus with a platform for the assembly of viral replication complexes, improving efficiency of RNA synthesis. The membranes observed in coronavirus infected cells include double membrane vesicles. By nature of their double membrane, these vesicles resemble cellular autophagosomes, generated during the cellular autophagy pathway. In addition, coronavirus infection has been demonstrated to induce autophagy. Here we review current knowledge of coronavirus induced membrane rearrangements and the involvement of autophagy or autophagy protein microtubule associated protein 1B light chain 3 (LC3) in coronavirus replication.', 'denotations': [{'obj': 'Disease:D018352', 'span': {'begin': 562, 'end': 583}}, {'obj': 'Gene:4131', 'span': {'begin': 761, 'end': 794}}, {'obj': 'Gene:84557', 'span': {'begin': 810, 'end': 813}}]}]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'sourceid': '22906110', 'sourcedb': 'PubMed', 'text': 'Stimulation with a class A CpG oligonucleotide enhances resistance to infection with feline viruses from five different families. Domestic cats are commonly affected by viral pathogens that induce lengthy infections with fatal outcomes. Prevention of viral propagation is of primordial importance in shelters and catteries, where cats from different backgrounds have narrow contacts. Oligonucleotides (ODN) containing cytosine-phosphate-guanosine motifs of class A (CpG-A) are highly potent synthetic inducers of innate antiviral mechanisms. The aim of this study was to test their ability to modulate innate immune responses and prevent viral replication as stand-alone agents in the domestic cat. CpG-A stimulation of feline peripheral blood mononuclear cells (PBMCs) enhanced their proliferation, increased the presence of co-stimulatory molecules on their surface and influenced their gene expression profiles in an antiviral orientation. Incubation of the supernatants of CpG-A stimulated PBMCs with feline cell lines of epithelial and fibroblastic origin induced expression of the antiviral myxovirus resistance (Mx) gene in these target cells, which also showed enhanced resistance to feline viruses from five distinct families, namely Coronaviridae, Herpesviridae, Caliciviridae, Parvoviridae, and Retroviridae. Most importantly, subcutaneous administration of CpG-A in domestic cats systemically increased the expression of Mx, reaching maximal levels within 24 h. Plasma from treated cats could furthermore inhibit viral replication in vitro. Altogether, our data highlight the promising potential of CpG-A to induce a preventive antiviral state in the cat and to protect feline populations against a broad range of virus infections.', 'denotations': [{'obj': 'Disease:D007239', 'span': {'begin': 197, 'end': 215}}, {'obj': 'Disease:D015658', 'span': {'begin': 1711, 'end': 1742}}, {'obj': 'Chemical:', 'span': {'begin': 1611, 'end': 1616}}, {'obj': 'Chemical:', 'span': {'begin': 1369, 'end': 1374}}, {'obj': 'Chemical:', 'span': {'begin': 977, 'end': 982}}, {'obj': 'Chemical:', 'span': {'begin': 699, 'end': 704}}, {'obj': 'Chemical:', 'span': {'begin': 466, 'end': 471}}, {'obj': 'Chemical:D006151', 'span': {'begin': 437, 'end': 446}}, {'obj': 'Chemical:CHEBI:26020', 'span': {'begin': 427, 'end': 436}}, {'obj': 'Chemical:D003596', 'span': {'begin': 418, 'end': 426}}, {'obj': 'Chemical:', 'span': {'begin': 27, 'end': 30}}, {'obj': 'Species:9685', 'span': {'begin': 130, 'end': 143}}, {'obj': 'Species:9685', 'span': {'begin': 330, 'end': 334}}, {'obj': 'Species:9685', 'span': {'begin': 685, 'end': 697}}, {'obj': 'Species:9685', 'span': {'begin': 1378, 'end': 1391}}, {'obj': 'Species:9685', 'span': {'begin': 1494, 'end': 1498}}]}]\n",
      "[{'sourceid': '22960954', 'sourcedb': 'PubMed', 'text': 'A novel double antibody sandwich-lateral flow immunoassay for the rapid and simple detection of hepatitis C virus. The objective of this study was to screen for antigens of the hepatitis C virus (HCV) to establish a new double antibody sandwich-lateral flow immunoassay (DAS-LFIA) method for testing the presence of anti-HCV antibodies in human serum or plasma. A series of different recombinant HCV proteins in Escherichia coli cells were constructed, expressed, purified and the new DAS-LFIA strip was developed. The sensitivity and specificity of new the DAS-LFIA strip were evaluated by detecting 23 HCV-positive sera, a set of quality control references for anti-HCV detection that contain known amounts of anti-HCV antibodies, and 8 HCV-negative sera. A total of 300 clinical serum samples was examined by both the new DAS-LFIA strip and enzyme-linked immunosorbent assay (ELISA). Data were analyzed using SPSS 11.5 software. The sensitivity and specificity of the new DAS-LFIA strip were 100%. The lowest test line of the HCV DAS-LFIA strips was 2 NCU/ml. Additionally, the concordance between the new DAS-LFIA strip and ELISA methods was 94.33%. In conclusion, our new testing method is rapid, simple, sensitive and specifically detects the presence of anti-HCV antibodies in human serum or plasma. Therefore, it may be used for monitoring HCV.', 'denotations': [{'obj': 'Disease:D006526', 'span': {'begin': 177, 'end': 194}}, {'obj': 'Disease:D006526', 'span': {'begin': 76, 'end': 113}}, {'obj': 'Disease:D006526', 'span': {'begin': 196, 'end': 199}}, {'obj': 'Disease:D006526', 'span': {'begin': 321, 'end': 324}}, {'obj': 'Disease:D006526', 'span': {'begin': 396, 'end': 399}}, {'obj': 'Disease:D006526', 'span': {'begin': 604, 'end': 607}}, {'obj': 'Disease:D006526', 'span': {'begin': 668, 'end': 671}}, {'obj': 'Disease:D006526', 'span': {'begin': 717, 'end': 720}}, {'obj': 'Disease:D006526', 'span': {'begin': 739, 'end': 742}}, {'obj': 'Disease:D006526', 'span': {'begin': 1029, 'end': 1032}}, {'obj': 'Disease:D006526', 'span': {'begin': 1266, 'end': 1269}}, {'obj': 'Disease:D006526', 'span': {'begin': 1348, 'end': 1351}}, {'obj': 'Species:562', 'span': {'begin': 412, 'end': 428}}]}]\n",
      "[{'sourceid': '23351276', 'sourcedb': 'PubMed', 'text': 'ciliaFA: a research tool for automated, high-throughput measurement of ciliary beat frequency using freely available software. BACKGROUND: Analysis of ciliary function for assessment of patients suspected of primary ciliary dyskinesia (PCD) and for research studies of respiratory and ependymal cilia requires assessment of both ciliary beat pattern and beat frequency. While direct measurement of beat frequency from high-speed video recordings is the most accurate and reproducible technique it is extremely time consuming. The aim of this study was to develop a freely available automated method of ciliary beat frequency analysis from digital video (AVI) files that runs on open-source software (ImageJ) coupled to Microsoft Excel, and to validate this by comparison to the direct measuring high-speed video recordings of respiratory and ependymal cilia. These models allowed comparison to cilia beating between 3 and 52   Hz. METHODS: Digital video files of motile ciliated ependymal (frequency range 34 to 52   Hz) and respiratory epithelial cells (frequency 3 to 18   Hz) were captured using a high-speed digital video recorder. To cover the range above between 18 and 37   Hz the frequency of ependymal cilia were slowed by the addition of the pneumococcal toxin pneumolysin. Measurements made directly by timing a given number of individual ciliary beat cycles were compared with those obtained using the automated ciliaFA system. RESULTS: The overall mean difference (   SD) between the ciliaFA and direct measurement high-speed digital imaging methods was -0.05        1.25   Hz, the correlation coefficient was shown to be 0.991 and the Bland-Altman limits of agreement were from -1.99 to 1.49   Hz for respiratory and from -2.55 to 3.25   Hz for ependymal cilia. CONCLUSIONS: A plugin for ImageJ was developed that extracts pixel intensities and performs fast Fourier transformation (FFT) using Microsoft Excel. The ciliaFA software allowed automated, high throughput measurement of respiratory and ependymal ciliary beat frequency (range 3 to 52   Hz) and avoids operator error due to selection bias. We have included free access to the ciliaFA plugin and installation instructions in Additional file 1 accompanying this manuscript that other researchers may use.', 'denotations': [{'obj': 'Disease:D004409', 'span': {'begin': 208, 'end': 234}}, {'obj': 'Disease:D004409', 'span': {'begin': 236, 'end': 239}}, {'obj': 'Species:9606', 'span': {'begin': 186, 'end': 194}}]}]\n",
      "[{'sourceid': '23710205', 'sourcedb': 'PubMed', 'text': 'New insights in recurrent HCV infection after liver transplantation. Hepatitis C virus (HCV) is a small-enveloped RNA virus belonging to the Flaviviridae family. Since first identified in 1989, HCV has been estimated to infect 170 million people worldwide. Mostly chronic hepatitis C virus has a uniform natural history, from liver cirrhosis to the development of hepatocellular carcinoma. The current therapy for HCV infection consists of a combination of Pegylated interferon and ribavirin. On the other hand, HCV-related liver disease is also the leading indication for liver transplantation. However, posttransplant HCV re-infection of the graft has been reported to be universal. Furthermore, the graft after HCV re-infection often results in accelerated progression to liver failure. In addition, treatment of recurrent HCV infection after liver transplantation is often compromised by enhanced adverse effects and limited efficacy of interferon-based therapies. Taken together, poor outcome after HCV re-infection, regardless of grafts or recipients, poses a major issue for the hepatologists and transplant surgeons. The aim of this paper is to review several specific aspects regarding HCV re-infection after transplant: risk factors, current therapeutics for HCV in different stages of liver transplantation, cellular function of HCV proteins, and molecular mechanisms of HCV entry. Hopefully, this paper will inspire new strategies and novel inhibitors against recurrent HCV infection after liver transplantation and greatly improve its overall outcome.', 'denotations': [{'obj': 'Disease:D006526', 'span': {'begin': 69, 'end': 86}}, {'obj': 'Disease:D006526', 'span': {'begin': 88, 'end': 91}}, {'obj': 'Disease:D006526', 'span': {'begin': 26, 'end': 39}}, {'obj': 'Disease:D006526', 'span': {'begin': 194, 'end': 197}}, {'obj': 'Disease:D019698', 'span': {'begin': 257, 'end': 289}}, {'obj': 'Disease:D005355', 'span': {'begin': 332, 'end': 341}}, {'obj': 'Disease:D006528', 'span': {'begin': 364, 'end': 388}}, {'obj': 'Disease:D006526', 'span': {'begin': 414, 'end': 427}}, {'obj': 'Disease:D006526', 'span': {'begin': 512, 'end': 515}}, {'obj': 'Disease:D008107', 'span': {'begin': 524, 'end': 537}}, {'obj': 'Disease:D006526', 'span': {'begin': 620, 'end': 636}}, {'obj': 'Disease:D006526', 'span': {'begin': 714, 'end': 730}}, {'obj': 'Disease:D017093', 'span': {'begin': 775, 'end': 788}}, {'obj': 'Disease:D006526', 'span': {'begin': 826, 'end': 839}}, {'obj': 'Disease:D006526', 'span': {'begin': 1004, 'end': 1020}}, {'obj': 'Disease:D006526', 'span': {'begin': 1195, 'end': 1211}}, {'obj': 'Disease:D006526', 'span': {'begin': 1269, 'end': 1272}}, {'obj': 'Disease:D006526', 'span': {'begin': 1340, 'end': 1343}}, {'obj': 'Disease:D006526', 'span': {'begin': 1382, 'end': 1385}}, {'obj': 'Disease:D006526', 'span': {'begin': 1482, 'end': 1495}}, {'obj': 'Chemical:D012254', 'span': {'begin': 482, 'end': 491}}, {'obj': 'Species:9606', 'span': {'begin': 239, 'end': 245}}]}]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'sourceid': '23936115', 'sourcedb': 'PubMed', 'text': 'Protective efficacy of VP1-specific neutralizing antibody associated with a reduction of viral load and pro-inflammatory cytokines in human SCARB2-transgenic mice. Hand-foot-mouth diseases (HFMD) caused by enterovirus 71 (EV71) and coxsackievirus 16 (CVA16) in children have now become a severe public health issue in the Asian-Pacific region. Recently we have successfully developed transgenic mice expressing human scavenger receptor class B member 2 (hSCARB2, a receptor of EV71 and CVA16) as an animal model for evaluating the pathogenesis of enterovirus infections. In this study, hSCARB2-transgenic mice were used to investigate the efficacy conferred by a previously described EV71 neutralizing antibody, N3. A single injection of N3 effectively inhibited the HFMD-like skin scurfs in mice pre-infected with clinical isolate of EV71 E59 (B4 genotype) or prevented severe limb paralysis and death in mice pre-inoculated with 5746 (C2 genotype). This protection was correlated with remarkable reduction of viral loads in the brain, spinal cord and limb muscles. Accumulated viral loads and the associated pro-inflammatory cytokines were all reduced. The protective efficacy of N3 was not observed in animals challenged with CVA16. This could be due to dissimilarity sequences of the neutralizing epitope found in CVA16. These results indicate N3 could be useful in treating severe EV71 infections and the hSCARB2-transgenic mouse could be used to evaluate the protective efficacy of potential anti-enterovirus agent candidates.', 'denotations': [{'obj': 'Disease:D006232', 'span': {'begin': 164, 'end': 188}}, {'obj': 'Disease:D006232', 'span': {'begin': 190, 'end': 194}}, {'obj': 'Disease:D004769', 'span': {'begin': 547, 'end': 569}}, {'obj': 'Disease:D006232', 'span': {'begin': 767, 'end': 771}}, {'obj': 'Disease:D010243', 'span': {'begin': 883, 'end': 892}}, {'obj': 'Disease:D003643', 'span': {'begin': 897, 'end': 902}}, {'obj': 'Chemical:', 'span': {'begin': 738, 'end': 740}}, {'obj': 'Gene:950', 'span': {'begin': 140, 'end': 146}}, {'obj': 'Gene:950', 'span': {'begin': 417, 'end': 452}}, {'obj': 'Gene:950', 'span': {'begin': 454, 'end': 461}}, {'obj': 'Gene:950', 'span': {'begin': 586, 'end': 593}}, {'obj': 'Gene:950', 'span': {'begin': 1410, 'end': 1417}}, {'obj': 'Species:9606', 'span': {'begin': 261, 'end': 269}}, {'obj': 'Species:9606', 'span': {'begin': 134, 'end': 139}}, {'obj': 'Species:10090', 'span': {'begin': 147, 'end': 162}}, {'obj': 'Species:10090', 'span': {'begin': 384, 'end': 399}}, {'obj': 'Species:9606', 'span': {'begin': 411, 'end': 416}}, {'obj': 'Species:10090', 'span': {'begin': 594, 'end': 609}}, {'obj': 'Species:10090', 'span': {'begin': 792, 'end': 796}}, {'obj': 'Species:10090', 'span': {'begin': 906, 'end': 910}}, {'obj': 'Species:10090', 'span': {'begin': 1429, 'end': 1434}}]}]\n",
      "[{'sourceid': '24223640', 'sourcedb': 'PubMed', 'text': 'Serum proteomics analysis and comparisons using iTRAQ in the progression of hepatitis B. The aim of this study was to analyze the changes in serum protein levels in the progression of hepatitis B using isobaric tags for relative and absolute quantitation (iTRAQ) analysis, in addition to comparing the serum protein levels of patients with chronic hepatitis B (CHB), patients with hepatitis B virus-induced acute-on-chronic liver failure (HBV-induced ACLF) and normal individuals. Protein analysis was performed on 15 serum samples using iTRAQ. The study population included healthy controls (n=5), patients with CHB (n=5) and patients with HBV-induced ACLF (n=5). Western blotting was used to verify the results in an additional nine serum samples from healthy controls, patients with CHB and patients with HBV-induced ACLF (n=3, respectively). Using iTRAQ analysis, 16 different serum proteins with >= 1.5-fold differences in expression levels were identified in the patients with CHB and ACLF compared with the healthy controls. Five of those proteins, C-reactive protein precursor, hemoglobin b chain variant Hb S-Wake, apolipoprotein J precursor, platelet factor 4 precursor and vitronectin, which demonstrated the greatest differences in their expression levels and the most significant correlation with liver diseases, were subsequently verified using western blotting. The western blotting results were consistent with the results from the iTRAQ. Two of the five proteins are not classified by biological process, and the biological functions of all the proteins in HBV-induced ACLF remain unclear. This preliminary study demonstrated that a correlation between the expression of various serum proteins and the different pathogenetic conditions induced by HBV may exist. The analysis of a larger number of samples is required to identify potential protein biomarkers that may be involved in the pathogenesis and progression of hepatitis B.', 'denotations': [{'obj': 'Disease:D006509', 'span': {'begin': 76, 'end': 87}}, {'obj': 'Disease:D006509', 'span': {'begin': 184, 'end': 195}}, {'obj': 'Disease:D019694', 'span': {'begin': 340, 'end': 359}}, {'obj': 'Disease:D019694', 'span': {'begin': 361, 'end': 364}}, {'obj': 'Disease:D006509', 'span': {'begin': 381, 'end': 392}}, {'obj': 'Disease:D017114', 'span': {'begin': 407, 'end': 437}}, {'obj': 'Disease:D065290', 'span': {'begin': 451, 'end': 455}}, {'obj': 'Disease:D019694', 'span': {'begin': 613, 'end': 616}}, {'obj': 'Disease:D065290', 'span': {'begin': 653, 'end': 657}}, {'obj': 'Disease:D019694', 'span': {'begin': 786, 'end': 789}}, {'obj': 'Disease:D065290', 'span': {'begin': 820, 'end': 824}}, {'obj': 'Disease:D019694', 'span': {'begin': 983, 'end': 986}}, {'obj': 'Disease:D065290', 'span': {'begin': 991, 'end': 995}}, {'obj': 'Disease:D008107', 'span': {'begin': 1310, 'end': 1324}}, {'obj': 'Disease:D065290', 'span': {'begin': 1586, 'end': 1590}}, {'obj': 'Disease:D006509', 'span': {'begin': 1935, 'end': 1946}}, {'obj': 'Species:9606', 'span': {'begin': 326, 'end': 334}}, {'obj': 'Species:9606', 'span': {'begin': 367, 'end': 375}}, {'obj': 'Species:10407', 'span': {'begin': 439, 'end': 442}}, {'obj': 'Species:9606', 'span': {'begin': 599, 'end': 607}}, {'obj': 'Species:9606', 'span': {'begin': 627, 'end': 635}}, {'obj': 'Species:10407', 'span': {'begin': 641, 'end': 644}}, {'obj': 'Species:9606', 'span': {'begin': 772, 'end': 780}}, {'obj': 'Species:9606', 'span': {'begin': 794, 'end': 802}}, {'obj': 'Species:10407', 'span': {'begin': 808, 'end': 811}}, {'obj': 'Species:9606', 'span': {'begin': 969, 'end': 977}}, {'obj': 'Species:10407', 'span': {'begin': 1574, 'end': 1577}}, {'obj': 'Species:10407', 'span': {'begin': 1764, 'end': 1767}}]}]\n",
      "[{'sourceid': '24278493', 'sourcedb': 'PubMed', 'text': 'Ultra-deep sequencing of intra-host rabies virus populations during cross-species transmission. One of the hurdles to understanding the role of viral quasispecies in RNA virus cross-species transmission (CST) events is the need to analyze a densely sampled outbreak using deep sequencing in order to measure the amount of mutation occurring on a small time scale. In 2009, the California Department of Public Health reported a dramatic increase (350) in the number of gray foxes infected with a rabies virus variant for which striped skunks serve as a reservoir host in Humboldt County. To better understand the evolution of rabies, deep-sequencing was applied to 40 unpassaged rabies virus samples from the Humboldt outbreak. For each sample, approximately 11 kb of the 12 kb genome was amplified and sequenced using the Illumina platform. Average coverage was 17,448 and this allowed characterization of the rabies virus population present in each sample at unprecedented depths. Phylogenetic analysis of the consensus sequence data demonstrated that samples clustered according to date (1995 vs. 2009) and geographic location (northern vs. southern). A single amino acid change in the G protein distinguished a subset of northern foxes from a haplotype present in both foxes and skunks, suggesting this mutation may have played a role in the observed increased transmission among foxes in this region. Deep-sequencing data indicated that many genetic changes associated with the CST event occurred prior to 2009 since several nonsynonymous mutations that were present in the consensus sequences of skunk and fox rabies samples obtained from 20032010 were present at the sub-consensus level (as rare variants in the viral population) in skunk and fox samples from 1995. These results suggest that analysis of rare variants within a viral population may yield clues to ancestral genomes and identify rare variants that have the potential to be selected for if environment conditions change.', 'denotations': [{'obj': 'Chemical:CHEBI:33704', 'span': {'begin': 1163, 'end': 1173}}, {'obj': 'Species:11292', 'span': {'begin': 36, 'end': 48}}, {'obj': 'Species:11292', 'span': {'begin': 495, 'end': 515}}, {'obj': 'Species:30548', 'span': {'begin': 526, 'end': 540}}, {'obj': 'Species:11292', 'span': {'begin': 678, 'end': 690}}, {'obj': 'Species:11292', 'span': {'begin': 910, 'end': 922}}]}]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'sourceid': '24278022', 'sourcedb': 'PubMed', 'text': 'The Neonatal Fc receptor (FcRn) enhances human immunodeficiency virus type 1 (HIV-1) transcytosis across epithelial cells. The mechanisms by which human immunodeficiency virus type 1 (HIV-1) crosses mucosal surfaces to establish infection are unknown. Acidic genital secretions of HIV-1-infected women contain HIV-1 likely coated by antibody. We found that the combination of acidic pH and Env-specific IgG, including that from cervicovaginal and seminal fluids of HIV-1-infected individuals, augmented transcytosis across epithelial cells as much as 20-fold compared with Env-specific IgG at neutral pH or non-specific IgG at either pH. Enhanced transcytosis was observed with clinical HIV-1 isolates, including transmitted/founder strains, and was eliminated in Fc neonatal receptor (FcRn)-knockdown epithelial cells. Non-neutralizing antibodies allowed similar or less transcytosis than neutralizing antibodies. However, the ratio of total:infectious virus was higher for neutralizing antibodies, indicating that they allowed transcytosis while blocking infectivity of transcytosed virus. Immunocytochemistry revealed abundant FcRn expression in columnar epithelia lining the human endocervix and penile urethra. Acidity and Env-specific IgG enhance transcytosis of virus across epithelial cells via FcRn and could facilitate translocation of virus to susceptible target cells following sexual exposure.', 'denotations': [{'obj': 'Gene:2217', 'span': {'begin': 26, 'end': 30}}, {'obj': 'Gene:155971', 'span': {'begin': 390, 'end': 393}}, {'obj': 'Gene:155971', 'span': {'begin': 573, 'end': 576}}, {'obj': 'Gene:2217', 'span': {'begin': 786, 'end': 790}}, {'obj': 'Gene:2217', 'span': {'begin': 1130, 'end': 1134}}, {'obj': 'Gene:155971', 'span': {'begin': 1228, 'end': 1231}}, {'obj': 'Gene:2217', 'span': {'begin': 1303, 'end': 1307}}, {'obj': 'Species:11676', 'span': {'begin': 147, 'end': 189}}, {'obj': 'Species:11676', 'span': {'begin': 41, 'end': 83}}, {'obj': 'Species:11676', 'span': {'begin': 281, 'end': 286}}, {'obj': 'Species:9606', 'span': {'begin': 296, 'end': 301}}, {'obj': 'Species:11676', 'span': {'begin': 310, 'end': 315}}, {'obj': 'Species:11676', 'span': {'begin': 465, 'end': 470}}, {'obj': 'Species:11676', 'span': {'begin': 687, 'end': 692}}, {'obj': 'Species:12081', 'span': {'begin': 954, 'end': 959}}, {'obj': 'Species:12081', 'span': {'begin': 1085, 'end': 1090}}, {'obj': 'Species:9606', 'span': {'begin': 1179, 'end': 1184}}, {'obj': 'Species:12081', 'span': {'begin': 1269, 'end': 1274}}, {'obj': 'Species:12081', 'span': {'begin': 1346, 'end': 1351}}]}]\n",
      "[{'sourceid': '24586153', 'sourcedb': 'PubMed', 'text': 'Unifying viral genetics and human transportation data to predict the global transmission dynamics of human influenza H3N2. Information on global human movement patterns is central to spatial epidemiological models used to predict the behavior of influenza and other infectious diseases. Yet it remains difficult to test which modes of dispersal drive pathogen spread at various geographic scales using standard epidemiological data alone. Evolutionary analyses of pathogen genome sequences increasingly provide insights into the spatial dynamics of influenza viruses, but to date they have largely neglected the wealth of information on human mobility, mainly because no statistical framework exists within which viral gene sequences and empirical data on host movement can be combined. Here, we address this problem by applying a phylogeographic approach to elucidate the global spread of human influenza subtype H3N2 and assess its ability to predict the spatial spread of human influenza A viruses worldwide. Using a framework that estimates the migration history of human influenza while simultaneously testing and quantifying a range of potential predictive variables of spatial spread, we show that the global dynamics of influenza H3N2 are driven by air passenger flows, whereas at more local scales spread is also determined by processes that correlate with geographic distance. Our analyses further confirm a central role for mainland China and Southeast Asia in maintaining a source population for global influenza diversity. By comparing model output with the known pandemic expansion of H1N1 during 2009, we demonstrate that predictions of influenza spatial spread are most accurate when data on human mobility and viral evolution are integrated. In conclusion, the global dynamics of influenza viruses are best explained by combining human mobility data with the spatial information inherent in sampled viral genomes. The integrated approach introduced here offers great potential for epidemiological surveillance through phylogeographic reconstructions and for improving predictive models of disease control.', 'denotations': [{'obj': 'Disease:D007251', 'span': {'begin': 107, 'end': 116}}, {'obj': 'Disease:D007251', 'span': {'begin': 246, 'end': 255}}, {'obj': 'Disease:D003141', 'span': {'begin': 266, 'end': 285}}, {'obj': 'Disease:D007251', 'span': {'begin': 549, 'end': 558}}, {'obj': 'Disease:D007251', 'span': {'begin': 896, 'end': 918}}, {'obj': 'Disease:D007251', 'span': {'begin': 981, 'end': 990}}, {'obj': 'Disease:D007251', 'span': {'begin': 1076, 'end': 1085}}, {'obj': 'Disease:D007251', 'span': {'begin': 1228, 'end': 1237}}, {'obj': 'Disease:D007251', 'span': {'begin': 1515, 'end': 1524}}, {'obj': 'Disease:D007251', 'span': {'begin': 1652, 'end': 1661}}, {'obj': 'Disease:D007251', 'span': {'begin': 1797, 'end': 1806}}, {'obj': 'Species:9606', 'span': {'begin': 145, 'end': 150}}, {'obj': 'Species:9606', 'span': {'begin': 28, 'end': 33}}, {'obj': 'Species:9606', 'span': {'begin': 101, 'end': 106}}, {'obj': 'Species:9606', 'span': {'begin': 637, 'end': 642}}, {'obj': 'Species:9606', 'span': {'begin': 890, 'end': 895}}, {'obj': 'Species:9606', 'span': {'begin': 975, 'end': 980}}, {'obj': 'Species:9606', 'span': {'begin': 1070, 'end': 1075}}, {'obj': 'Species:9606', 'span': {'begin': 1708, 'end': 1713}}, {'obj': 'Species:9606', 'span': {'begin': 1847, 'end': 1852}}]}]\n",
      "[{'sourceid': '24638205', 'sourcedb': 'PubMed', 'text': 'Hantavirus immunology of rodent reservoirs: current status and future directions. Hantaviruses are hosted by rodents, insectivores and bats. Several rodent-borne hantaviruses cause two diseases that share many features in humans, hemorrhagic fever with renal syndrome in Eurasia or hantavirus cardiopulmonary syndrome in the Americas. It is thought that the immune response plays a significant contributory role in these diseases. However, in reservoir hosts that have been closely examined, little or no pathology occurs and infection is persistent despite evidence of adaptive immune responses. Because most hantavirus reservoirs are not model organisms, it is difficult to conduct meaningful experiments that might shed light on how the viruses evade sterilizing immune responses and why immunopathology does not occur. Despite these limitations, recent advances in instrumentation and bioinformatics will have a dramatic impact on understanding reservoir host responses to hantaviruses by employing a systems biology approach to identify important pathways that mediate virus/reservoir relationships.', 'denotations': [{'obj': 'Disease:D006478', 'span': {'begin': 230, 'end': 247}}, {'obj': 'Disease:D007674', 'span': {'begin': 253, 'end': 267}}, {'obj': 'Disease:D006323', 'span': {'begin': 293, 'end': 317}}, {'obj': 'Disease:D007239', 'span': {'begin': 526, 'end': 535}}, {'obj': 'Species:9606', 'span': {'begin': 222, 'end': 228}}]}]\n",
      "[{'sourceid': '24816104', 'sourcedb': 'PubMed', 'text': 'Structural basis of sialidase in complex with geranylated flavonoids as potent natural inhibitors. Sialidase catalyzes the removal of a terminal sialic acid from glycoconjugates and plays a pivotal role in nutrition, cellular interactions and pathogenesis mediating various infectious diseases including cholera, influenza and sepsis. An array of antiviral sialidase agents have been developed and are commercially available, such as zanamivir and oseltamivir for treating influenza. However, the development of bacterial sialidase inhibitors has been much less successful. Here, natural polyphenolic geranylated flavonoids which show significant inhibitory effects against Cp-NanI, a sialidase from Clostridium perfringens, are reported. This bacterium causes various gastrointestinal diseases. The crystal structure of the Cp-NanI catalytic domain in complex with the best inhibitor, diplacone, is also presented. This structure explains how diplacone generates a stable enzyme-inhibitor complex. These results provide a structural framework for understanding the interaction between sialidase and natural flavonoids, which are promising scaffolds on which to discover new anti-sialidase agents.', 'denotations': [{'obj': 'Disease:D003141', 'span': {'begin': 274, 'end': 293}}, {'obj': 'Disease:D002771', 'span': {'begin': 304, 'end': 311}}, {'obj': 'Disease:D007251', 'span': {'begin': 313, 'end': 322}}, {'obj': 'Disease:D018805', 'span': {'begin': 327, 'end': 333}}, {'obj': 'Disease:D007251', 'span': {'begin': 473, 'end': 482}}, {'obj': 'Disease:D005767', 'span': {'begin': 769, 'end': 794}}, {'obj': 'Chemical:D005419', 'span': {'begin': 1108, 'end': 1118}}, {'obj': 'Chemical:', 'span': {'begin': 886, 'end': 895}}, {'obj': 'Chemical:', 'span': {'begin': 674, 'end': 681}}, {'obj': 'Chemical:', 'span': {'begin': 588, 'end': 623}}, {'obj': 'Chemical:D053139', 'span': {'begin': 448, 'end': 459}}, {'obj': 'Chemical:D053243', 'span': {'begin': 434, 'end': 443}}, {'obj': 'Chemical:D005419', 'span': {'begin': 58, 'end': 68}}, {'obj': 'Chemical:D019158', 'span': {'begin': 145, 'end': 156}}, {'obj': 'Species:1502', 'span': {'begin': 700, 'end': 723}}]}]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'sourceid': '24824328', 'sourcedb': 'PubMed', 'text': 'Usefulness of cellular analysis of bronchoalveolar lavage fluid for predicting the etiology of pneumonia in critically ill patients. BACKGROUND: The usefulness of bronchoalveolar lavage (BAL) fluid cellular analysis in pneumonia has not been adequately evaluated. This study investigated the ability of cellular analysis of BAL fluid to differentially diagnose bacterial pneumonia from viral pneumonia in adult patients who are admitted to intensive care unit. METHODS: BAL fluid cellular analysis was evaluated in 47 adult patients who underwent bronchoscopic BAL following less than 24 hours of antimicrobial agent exposure. The abilities of BAL fluid total white blood cell (WBC) counts and differential cell counts to differentiate between bacterial and viral pneumonia were evaluated using receiver operating characteristic (ROC) curve analysis. RESULTS: Bacterial pneumonia (n=24) and viral pneumonia (n=23) were frequently associated with neutrophilic pleocytosis in BAL fluid. BAL fluid median total WBC count (2,815/  L vs. 300/  L, P<0.001) and percentage of neutrophils (80.5% vs. 54.0%, P=0.02) were significantly higher in the bacterial pneumonia group than in the viral pneumonia group. In ROC curve analysis, BAL fluid total WBC count showed the best discrimination, with an area under the curve of 0.855 (95% CI, 0.750-0.960). BAL fluid total WBC count >=  510/  L had a sensitivity of 83.3%, specificity of 78.3%, positive likelihood ratio (PLR) of 3.83, and negative likelihood ratio (NLR) of 0.21. When analyzed in combination with serum procalcitonin or C-reactive protein, sensitivity was 95.8%, specificity was 95.7%, PLR was 8.63, and NLR was 0.07. BAL fluid total WBC count >=  510/  L was an independent predictor of bacterial pneumonia with an adjusted odds ratio of 13.5 in multiple logistic regression analysis. CONCLUSIONS: Cellular analysis of BAL fluid can aid early differential diagnosis of bacterial pneumonia from viral pneumonia in critically ill patients.', 'denotations': [{'obj': 'Disease:D011014', 'span': {'begin': 219, 'end': 228}}, {'obj': 'Disease:D011014', 'span': {'begin': 95, 'end': 104}}, {'obj': 'Disease:D016638', 'span': {'begin': 108, 'end': 122}}, {'obj': 'Disease:D018410', 'span': {'begin': 361, 'end': 380}}, {'obj': 'Disease:D011024', 'span': {'begin': 386, 'end': 401}}, {'obj': 'Disease:D011024', 'span': {'begin': 744, 'end': 773}}, {'obj': 'Disease:D018410', 'span': {'begin': 860, 'end': 879}}, {'obj': 'Disease:D011024', 'span': {'begin': 891, 'end': 906}}, {'obj': 'Disease:D018410', 'span': {'begin': 1140, 'end': 1159}}, {'obj': 'Disease:D011024', 'span': {'begin': 1178, 'end': 1193}}, {'obj': 'Disease:D018410', 'span': {'begin': 1742, 'end': 1761}}, {'obj': 'Disease:D018410', 'span': {'begin': 1924, 'end': 1943}}, {'obj': 'Disease:D011024', 'span': {'begin': 1949, 'end': 1964}}, {'obj': 'Disease:D016638', 'span': {'begin': 1968, 'end': 1982}}, {'obj': 'Chemical:C029100', 'span': {'begin': 1557, 'end': 1570}}, {'obj': 'Gene:1401', 'span': {'begin': 1574, 'end': 1592}}, {'obj': 'Species:9606', 'span': {'begin': 123, 'end': 131}}, {'obj': 'Species:9606', 'span': {'begin': 411, 'end': 419}}, {'obj': 'Species:9606', 'span': {'begin': 524, 'end': 532}}, {'obj': 'Species:9606', 'span': {'begin': 1983, 'end': 1991}}]}]\n",
      "[{'sourceid': '24895650', 'sourcedb': 'PubMed', 'text': 'Comparative analysis of lycorine in wild plant and callus culture samples of Hymenocallis littoralis by HPLC-UV method. The Hymenocallis littoralis, an ornamental and medicinal plant, had been traditionally used for wound healing. In the present study, an analytical method using HPLC with ultraviolet detection was developed for the quantification of lycorine in the extracts of different parts of wild plant and tissue culture samples of H. littoralis. The separation was achieved using a reversed-phase column. The method was found to be accurate, repeatable, and sensitive for the quantification of minute amount of lycorine present in the samples. The highest lycorine content was found in the bulb extract (2.54    0.02  g/mg) whereas the least was in the root extract (0.71    0.02  g/mg) of the wild plants. Few callus culture samples had high content of lycorine, comparable to that of wild plants. The results showed that plant growth regulators, 2,4-dichlorophenoxyacetic acid (2,4-D) alone at 4.5  M (2.58    0.38  g/mg) or a combination of 2,4-D at 9.00  M with 4.5  M of 6-benzylaminopurine (BAP), were the optimum concentrations for the production of high lycorine (2.45    0.15  g/mg) content in callus culture. The present analytical method could be of value for routine quantification of lycorine in the tissue culture production and standardization of the raw material or extracts of H. littoralis.', 'denotations': [{'obj': 'Chemical:C015330', 'span': {'begin': 1306, 'end': 1314}}, {'obj': 'Chemical:C015330', 'span': {'begin': 1171, 'end': 1179}}, {'obj': 'Chemical:CHEBI:29022', 'span': {'begin': 1106, 'end': 1109}}, {'obj': 'Chemical:C000142', 'span': {'begin': 1085, 'end': 1104}}, {'obj': 'Chemical:D015084', 'span': {'begin': 1053, 'end': 1058}}, {'obj': 'Chemical:D015084', 'span': {'begin': 989, 'end': 994}}, {'obj': 'Chemical:D015084', 'span': {'begin': 957, 'end': 987}}, {'obj': 'Chemical:C015330', 'span': {'begin': 863, 'end': 871}}, {'obj': 'Chemical:C015330', 'span': {'begin': 665, 'end': 673}}, {'obj': 'Chemical:C015330', 'span': {'begin': 620, 'end': 628}}, {'obj': 'Chemical:C015330', 'span': {'begin': 352, 'end': 360}}, {'obj': 'Chemical:C015330', 'span': {'begin': 24, 'end': 32}}]}]\n",
      "[{'sourceid': '27713263', 'sourcedb': 'PubMed', 'text': 'Cell-Penetrating Peptides for Antiviral Drug Development. UNASSIGNED: Viral diseases affect hundreds of millions of people worldwide, and the few available drugs to treat these diseases often come with limitations. The key obstacle to the development of new antiviral agents is their delivery into infected cells in vivo. Cell-penetrating peptides (CPPs) are short peptides that can cross the cellular lipid bilayer with the remarkable capability to shuttle conjugated cargoes into cells. CPPs have been successfully utilized to enhance the cellular uptake and intracellular trafficking of antiviral molecules, and thereby increase the inhibitory activity of potential antiviral proteins and oligonucleotide analogues, both in cultured cells and in animal models. This review will address the notable findings of these studies, highlighting some promising results and discussing the challenges CPP technology has to overcome for further clinical applications.', 'denotations': [{'obj': 'Disease:D014777', 'span': {'begin': 70, 'end': 84}}]}]\n",
      "[{'sourceid': '24875882', 'sourcedb': 'PubMed', 'text': \"Rapid evolution of PARP genes suggests a broad role for ADP-ribosylation in host-virus conflicts. Post-translational protein modifications such as phosphorylation and ubiquitinylation are common molecular targets of conflict between viruses and their hosts. However, the role of other post-translational modifications, such as ADP-ribosylation, in host-virus interactions is less well characterized. ADP-ribosylation is carried out by proteins encoded by the PARP (also called ARTD) gene family. The majority of the 17 human PARP genes are poorly characterized. However, one PARP protein, PARP13/ZAP, has broad antiviral activity and has evolved under positive (diversifying) selection in primates. Such evolution is typical of domains that are locked in antagonistic 'arms races' with viral factors. To identify additional PARP genes that may be involved in host-virus interactions, we performed evolutionary analyses on all primate PARP genes to search for signatures of rapid evolution. Contrary to expectations that most PARP genes are involved in 'housekeeping' functions, we found that nearly one-third of PARP genes are evolving under strong recurrent positive selection. We identified a >300 amino acid disordered region of PARP4, a component of cytoplasmic vault structures, to be rapidly evolving in several mammalian lineages, suggesting this region serves as an important host-pathogen specificity interface. We also found positive selection of PARP9, 14 and 15, the only three human genes that contain both PARP domains and macrodomains. Macrodomains uniquely recognize, and in some cases can reverse, protein mono-ADP-ribosylation, and we observed strong signatures of recurrent positive selection throughout the macro-PARP macrodomains. Furthermore, PARP14 and PARP15 have undergone repeated rounds of gene birth and loss during vertebrate evolution, consistent with recurrent gene innovation. Together with previous studies that implicated several PARPs in immunity, as well as those that demonstrated a role for virally encoded macrodomains in host immune evasion, our evolutionary analyses suggest that addition, recognition and removal of ADP-ribosylation is a critical, underappreciated currency in host-virus conflicts.\", 'denotations': [{'obj': 'Disease:C564422', 'span': {'begin': 56, 'end': 72}}, {'obj': 'Disease:C564422', 'span': {'begin': 327, 'end': 343}}, {'obj': 'Disease:C564422', 'span': {'begin': 400, 'end': 416}}, {'obj': 'Disease:C564422', 'span': {'begin': 1628, 'end': 1644}}, {'obj': 'Disease:C564422', 'span': {'begin': 2158, 'end': 2174}}, {'obj': 'Chemical:CHEBI:33704', 'span': {'begin': 1200, 'end': 1210}}, {'obj': 'Gene:142', 'span': {'begin': 19, 'end': 23}}, {'obj': 'Gene:142', 'span': {'begin': 459, 'end': 463}}, {'obj': 'Gene:142', 'span': {'begin': 525, 'end': 529}}, {'obj': 'Gene:142', 'span': {'begin': 575, 'end': 579}}, {'obj': 'Gene:56829', 'span': {'begin': 589, 'end': 595}}, {'obj': 'Gene:56829', 'span': {'begin': 596, 'end': 599}}, {'obj': 'Gene:142', 'span': {'begin': 824, 'end': 828}}, {'obj': 'Gene:142', 'span': {'begin': 934, 'end': 938}}, {'obj': 'Gene:142', 'span': {'begin': 1025, 'end': 1029}}, {'obj': 'Gene:142', 'span': {'begin': 1112, 'end': 1116}}, {'obj': 'Gene:143', 'span': {'begin': 1232, 'end': 1237}}, {'obj': 'Gene:83666', 'span': {'begin': 1457, 'end': 1462}}, {'obj': 'Gene:142', 'span': {'begin': 1520, 'end': 1524}}, {'obj': 'Gene:142', 'span': {'begin': 1733, 'end': 1737}}, {'obj': 'Gene:54625', 'span': {'begin': 1765, 'end': 1771}}, {'obj': 'Gene:165631', 'span': {'begin': 1776, 'end': 1782}}, {'obj': 'Species:9606', 'span': {'begin': 519, 'end': 524}}, {'obj': 'Species:9606', 'span': {'begin': 1318, 'end': 1327}}, {'obj': 'Species:9606', 'span': {'begin': 1490, 'end': 1495}}]}]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'sourceid': '24884954', 'sourcedb': 'PubMed', 'text': 'A fast and robust iterative algorithm for prediction of RNA pseudoknotted secondary structures. BACKGROUND: Improving accuracy and efficiency of computational methods that predict pseudoknotted RNA secondary structures is an ongoing challenge. Existing methods based on free energy minimization tend to be very slow and are limited in the types of pseudoknots that they can predict. Incorporating known structural information can improve prediction accuracy; however, there are not many methods for prediction of pseudoknotted structures that can incorporate structural information as input. There is even less understanding of the relative robustness of these methods with respect to partial information. RESULTS: We present a new method, Iterative HFold, for pseudoknotted RNA secondary structure prediction. Iterative HFold takes as input a pseudoknot-free structure, and produces a possibly pseudoknotted structure whose energy is at least as low as that of any (density-2) pseudoknotted structure containing the input structure. Iterative HFold leverages strengths of earlier methods, namely the fast running time of HFold, a method that is based on the hierarchical folding hypothesis, and the energy parameters of HotKnots V2.0.Our experimental evaluation on a large data set shows that Iterative HFold is robust with respect to partial information, with average accuracy on pseudoknotted structures steadily increasing from roughly 54% to 79% as the user provides up to 40% of the input structure.Iterative HFold is much faster than HotKnots V2.0, while having comparable accuracy. Iterative HFold also has significantly better accuracy than IPknot on our HK-PK and IP-pk168 data sets. CONCLUSIONS: Iterative HFold is a robust method for prediction of pseudoknotted RNA secondary structures, whose accuracy with more than 5% information about true pseudoknot-free structures is better than that of IPknot, and with about 35% information about true pseudoknot-free structures compares well with that of HotKnots V2.0 while being significantly faster. Iterative HFold and all data used in this work are freely available at http://www.cs.ubc.ca/~hjabbari/software.php.', 'denotations': [{'obj': 'Disease:D028361', 'span': {'begin': 513, 'end': 537}}, {'obj': 'Disease:D028361', 'span': {'begin': 1382, 'end': 1406}}]}]\n",
      "[{'sourceid': '24916787', 'sourcedb': 'PubMed', 'text': 'Difference in immune response in vaccinated and unvaccinated Swedish individuals after the 2009 influenza pandemic. BACKGROUND: Previous exposures to flu and subsequent immune responses may impact on 2009/2010 pandemic flu vaccine responses and clinical symptoms upon infection with the 2009 pandemic H1N1 influenza strain. Qualitative and quantitative differences in humoral and cellular immune responses associated with the flu vaccination in 2009/2010 (pandemic H1N1 vaccine) and natural infection have not yet been described in detail. We designed a longitudinal study to examine influenza- (flu-) specific immune responses and the association between pre-existing flu responses, symptoms of influenza-like illness (ILI), impact of pandemic flu infection, and pandemic flu vaccination in a cohort of 2,040 individuals in Sweden in 2009-2010. METHODS: Cellular flu-specific immune responses were assessed by whole-blood antigen stimulation assay, and humoral responses by a single radial hemolysis test. RESULTS: Previous seasonal flu vaccination was associated with significantly lower flu-specific IFN-y responses (using a whole-blood assay) at study entry. Pandemic flu vaccination induced long-lived T-cell responses (measured by IFN-y production) to influenza A strains, influenza B strains, and the matrix (M1) antigen. In contrast, individuals with pandemic flu infection (PCR positive) exhibited increased flu-specific T-cell responses shortly after onset of ILI symptoms but the immune response decreased after the flu season (spring 2010). We identified non-pandemic-flu vaccinated participants without ILI symptoms who showed an IFN-y production profile similar to pandemic-flu infected participants, suggesting exposure without experiencing clinical symptoms. CONCLUSIONS: Strong and long-lived flu-M1 specific immune responses, defined by IFN-y production, in individuals after vaccination suggest that M1-responses may contribute to protective cellular immune responses. Silent flu infections appeared to be frequent in 2009/2010. The pandemic flu vaccine induced qualitatively and quantitatively different humoral and cellular immune responses as compared to infection with the 2009 H1N1 pandemic H1N1 influenza strain.', 'denotations': [{'obj': 'Disease:D007251', 'span': {'begin': 150, 'end': 153}}, {'obj': 'Disease:D007251', 'span': {'begin': 96, 'end': 105}}, {'obj': 'Disease:D007251', 'span': {'begin': 219, 'end': 222}}, {'obj': 'Disease:D007251', 'span': {'begin': 306, 'end': 315}}, {'obj': 'Disease:D007251', 'span': {'begin': 426, 'end': 429}}, {'obj': 'Disease:D007251', 'span': {'begin': 584, 'end': 593}}, {'obj': 'Disease:D007251', 'span': {'begin': 596, 'end': 599}}, {'obj': 'Disease:D007251', 'span': {'begin': 669, 'end': 672}}, {'obj': 'Disease:D007251', 'span': {'begin': 696, 'end': 718}}, {'obj': 'Disease:D007251', 'span': {'begin': 720, 'end': 723}}, {'obj': 'Disease:D007239', 'span': {'begin': 736, 'end': 758}}, {'obj': 'Disease:D007251', 'span': {'begin': 764, 'end': 788}}, {'obj': 'Disease:D007251', 'span': {'begin': 864, 'end': 867}}, {'obj': 'Disease:D007251', 'span': {'begin': 1034, 'end': 1037}}, {'obj': 'Disease:D007251', 'span': {'begin': 1090, 'end': 1093}}, {'obj': 'Disease:D007251', 'span': {'begin': 1163, 'end': 1187}}, {'obj': 'Disease:D007251', 'span': {'begin': 1258, 'end': 1267}}, {'obj': 'Disease:D007251', 'span': {'begin': 1279, 'end': 1288}}, {'obj': 'Disease:D007239', 'span': {'begin': 1359, 'end': 1381}}, {'obj': 'Disease:D007251', 'span': {'begin': 1417, 'end': 1420}}, {'obj': 'Disease:D007251', 'span': {'begin': 1470, 'end': 1473}}, {'obj': 'Disease:D007251', 'span': {'begin': 1527, 'end': 1530}}, {'obj': 'Disease:D007251', 'span': {'begin': 1580, 'end': 1583}}, {'obj': 'Disease:D007251', 'span': {'begin': 1616, 'end': 1619}}, {'obj': 'Disease:D007239', 'span': {'begin': 1679, 'end': 1700}}, {'obj': 'Disease:D007251', 'span': {'begin': 1810, 'end': 1813}}, {'obj': 'Disease:D007239', 'span': {'begin': 1988, 'end': 2009}}, {'obj': 'Disease:D007251', 'span': {'begin': 2061, 'end': 2064}}, {'obj': 'Disease:D007251', 'span': {'begin': 2220, 'end': 2229}}, {'obj': 'Gene:3458', 'span': {'begin': 1103, 'end': 1108}}, {'obj': 'Gene:3458', 'span': {'begin': 1237, 'end': 1242}}, {'obj': 'Gene:3458', 'span': {'begin': 1643, 'end': 1648}}, {'obj': 'Gene:3458', 'span': {'begin': 1855, 'end': 1860}}, {'obj': 'Species:9606', 'span': {'begin': 1595, 'end': 1607}}, {'obj': 'Species:9606', 'span': {'begin': 1701, 'end': 1713}}]}]\n",
      "[{'sourceid': '24968358', 'sourcedb': 'PubMed', 'text': 'A nucleic-acid hydrolyzing single chain antibody confers resistance to DNA virus infection in hela cells and C57BL/6 mice. Viral protein neutralizing antibodies have been developed but they are limited only to the targeted virus and are often susceptible to antigenic drift. Here, we present an alternative strategy for creating virus-resistant cells and animals by ectopic expression of a nucleic acid hydrolyzing catalytic 3D8 single chain variable fragment (scFv), which has both DNase and RNase activities. HeLa cells (SCH7072) expressing 3D8 scFv acquired significant resistance to DNA viruses. Virus challenging with Herpes simplex virus (HSV) in 3D8 scFv transgenic cells and fluorescence resonance energy transfer (FRET) assay based on direct DNA cleavage analysis revealed that the induced resistance in HeLa cells was acquired by the nucleic acid hydrolyzing catalytic activity of 3D8 scFv. In addition, pseudorabies virus (PRV) infection in WT C57BL/6 mice was lethal, whereas transgenic mice (STG90) that expressed high levels of 3D8 scFv mRNA in liver, muscle, and brain showed a 56% survival rate 5 days after PRV intramuscular infection. The antiviral effects against DNA viruses conferred by 3D8 scFv expression in HeLa cells as well as an in vivo mouse system can be attributed to the nuclease activity that inhibits viral genome DNA replication in the nucleus and/or viral mRNA translation in the cytoplasm. Our results demonstrate that the nucleic-acid hydrolyzing activity of 3D8 scFv confers viral resistance to DNA viruses in vitro in HeLa cells and in an in vivo mouse system.', 'denotations': [{'obj': 'Disease:D015658', 'span': {'begin': 600, 'end': 643}}, {'obj': 'Disease:D015658', 'span': {'begin': 914, 'end': 948}}, {'obj': 'Disease:D015658', 'span': {'begin': 1124, 'end': 1151}}, {'obj': 'Chemical:CHEBI:33696', 'span': {'begin': 1459, 'end': 1471}}, {'obj': 'Chemical:', 'span': {'begin': 523, 'end': 530}}, {'obj': 'Gene:652070', 'span': {'begin': 461, 'end': 465}}, {'obj': 'Gene:652070', 'span': {'begin': 547, 'end': 551}}, {'obj': 'Gene:652070', 'span': {'begin': 657, 'end': 661}}, {'obj': 'Gene:652070', 'span': {'begin': 895, 'end': 899}}, {'obj': 'Gene:652070', 'span': {'begin': 1046, 'end': 1050}}, {'obj': 'Gene:652070', 'span': {'begin': 1212, 'end': 1216}}, {'obj': 'Gene:652070', 'span': {'begin': 1500, 'end': 1504}}, {'obj': 'Species:10090', 'span': {'begin': 117, 'end': 121}}, {'obj': 'Species:10090', 'span': {'begin': 963, 'end': 967}}, {'obj': 'Species:10090', 'span': {'begin': 988, 'end': 1003}}, {'obj': 'Species:10090', 'span': {'begin': 1264, 'end': 1269}}, {'obj': 'Species:10090', 'span': {'begin': 1586, 'end': 1591}}]}]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'sourceid': '25098254', 'sourcedb': 'PubMed', 'text': 'Effect of human movement on airborne disease transmission in an airplane cabin: study using numerical modeling and quantitative risk analysis. BACKGROUND: Airborne transmission of respiratory infectious disease in indoor environment (e.g. airplane cabin, conference room, hospital, isolated room and inpatient ward) may cause outbreaks of infectious diseases, which may lead to many infection cases and significantly influences on the public health. This issue has received more and more attentions from academics. This work investigates the influence of human movement on the airborne transmission of respiratory infectious diseases in an airplane cabin by using an accurate human model in numerical simulation and comparing the influences of different human movement behaviors on disease transmission. METHODS: The Eulerian-Lagrangian approach is adopted to simulate the dispersion and deposition of the expiratory aerosols. The dose-response model is used to assess the infection risks of the occupants. The likelihood analysis is performed as a hypothesis test on the input parameters and different human movement pattern assumptions. An in-flight SARS outbreak case is used for investigation. A moving person with different moving speeds is simulated to represent the movement behaviors. A digital human model was used to represent the detailed profile of the occupants, which was obtained by scanning a real thermal manikin using the 3D laser scanning system. RESULTS: The analysis results indicate that human movement can strengthen the downward transport of the aerosols, significantly reduce the overall deposition and removal rate of the suspended aerosols and increase the average infection risk in the cabin. The likelihood estimation result shows that the risk assessment results better fit the outcome of the outbreak case when the movements of the seated passengers are considered. The intake fraction of the moving person is significantly higher than most of the seated passengers. CONCLUSIONS: The infection risk distribution in the airplane cabin highly depends on the movement behaviors of the passengers and the index patient. The walking activities of the crew members and the seated passengers can significantly increase their personal infection risks. Taking the influence of the movement of the seated passengers and the index patient into consideration is necessary and important. For future studies, investigations on the behaviors characteristics of the passengers during flight will be useful and helpful for infection control.', 'denotations': [{'obj': 'Disease:D012131', 'span': {'begin': 180, 'end': 210}}, {'obj': 'Disease:D003141', 'span': {'begin': 339, 'end': 358}}, {'obj': 'Disease:D007239', 'span': {'begin': 383, 'end': 392}}, {'obj': 'Disease:D012131', 'span': {'begin': 602, 'end': 633}}, {'obj': 'Disease:D001523', 'span': {'begin': 760, 'end': 778}}, {'obj': 'Disease:D007239', 'span': {'begin': 973, 'end': 982}}, {'obj': 'Disease:D045169', 'span': {'begin': 1152, 'end': 1156}}, {'obj': 'Disease:D001523', 'span': {'begin': 1273, 'end': 1291}}, {'obj': 'Disease:D007239', 'span': {'begin': 1692, 'end': 1701}}, {'obj': 'Disease:D007239', 'span': {'begin': 2015, 'end': 2024}}, {'obj': 'Disease:D001523', 'span': {'begin': 2087, 'end': 2105}}, {'obj': 'Disease:D007239', 'span': {'begin': 2258, 'end': 2267}}, {'obj': 'Disease:D007239', 'span': {'begin': 2537, 'end': 2546}}, {'obj': 'Chemical:', 'span': {'begin': 2050, 'end': 2058}}, {'obj': 'Chemical:', 'span': {'begin': 640, 'end': 648}}, {'obj': 'Chemical:', 'span': {'begin': 239, 'end': 253}}, {'obj': 'Chemical:', 'span': {'begin': 64, 'end': 72}}, {'obj': 'Species:9606', 'span': {'begin': 10, 'end': 15}}, {'obj': 'Species:9606', 'span': {'begin': 555, 'end': 560}}, {'obj': 'Species:9606', 'span': {'begin': 676, 'end': 681}}, {'obj': 'Species:9606', 'span': {'begin': 754, 'end': 759}}, {'obj': 'Species:9606', 'span': {'begin': 1103, 'end': 1108}}, {'obj': 'Species:9606', 'span': {'begin': 1207, 'end': 1213}}, {'obj': 'Species:9606', 'span': {'begin': 1303, 'end': 1308}}, {'obj': 'Species:9606', 'span': {'begin': 1510, 'end': 1515}}, {'obj': 'Species:9606', 'span': {'begin': 1931, 'end': 1937}}, {'obj': 'Species:9606', 'span': {'begin': 2138, 'end': 2145}}, {'obj': 'Species:9606', 'span': {'begin': 2351, 'end': 2358}}]}]\n",
      "[{'sourceid': '25422070', 'sourcedb': 'PubMed', 'text': 'IFITM proteins are incorporated onto HIV-1 virion particles and negatively imprint their infectivity. BACKGROUND: Interferon induced transmembrane proteins 1, 2 and 3 (IFITMs) belong to a family of highly related antiviral factors that have been shown to interfere with a large spectrum of viruses including Filoviruses, Coronaviruses, Influenza virus, Dengue virus and HIV-1. In all these cases, the reported mechanism of antiviral inhibition indicates that the pool of IFITM proteins present in target cells blocks incoming viral particles in endosomal vesicles where they are subsequently degraded. RESULTS: In this study, we describe an additional mechanism through which IFITMs block HIV-1. In virus-producing cells, IFITMs coalesce with forming virions and are incorporated into viral particles. Expression of IFITMs during virion assembly leads to the production of virion particles of decreased infectivity that are mostly affected during entry in target cells. This mechanism of inhibition is exerted against different retroviruses and does not seem to be dependent on the type of Envelope present on retroviral particles. CONCLUSIONS: The results described here identify a novel mechanism through which IFITMs affect HIV-1 infectivity during the late phases of the viral life cycle. Put in the context of data obtained by other laboratories, these results indicate that IFITMs can target HIV at two distinct moments of its life cycle, in target cells as well as in virus-producing cells. These results raise the possibility that IFITMs could similarly affect distinct steps of the life cycle of a number of other viruses.', 'denotations': [{'obj': 'Disease:D015658', 'span': {'begin': 37, 'end': 49}}, {'obj': 'Disease:D015658', 'span': {'begin': 346, 'end': 375}}, {'obj': 'Species:11676', 'span': {'begin': 689, 'end': 694}}, {'obj': 'Species:11676', 'span': {'begin': 1227, 'end': 1232}}, {'obj': 'Species:11709', 'span': {'begin': 1398, 'end': 1401}}]}]\n",
      "[{'sourceid': '25470665', 'sourcedb': 'PubMed', 'text': 'Sparse evidence of MERS-CoV infection among animal workers living in Southern Saudi Arabia during 2012. Middle East respiratory syndrome coronavirus (MERS-CoV) is an emerging viral pathogen that primarily causes respiratory illness. We conducted a seroprevalence study of banked human serum samples collected in 2012 from Southern Saudi Arabia. Sera from 300 animal workers (17% with daily camel exposure) and 50 non-animal-exposed controls were examined for serological evidence of MERS-CoV infection by a pseudoparticle MERS-CoV spike protein neutralization assay.  None of the sera reproducibly neutralized the MERS-CoV-pseudotyped lentiviral vector. These data suggest that serological evidence of zoonotic transmission of MERS-CoV was not common among animal workers in Southern Saudi Arabia during July 2012.', 'denotations': [{'obj': 'Disease:D012131', 'span': {'begin': 212, 'end': 231}}, {'obj': 'Species:1335626', 'span': {'begin': 104, 'end': 158}}, {'obj': 'Species:1335626', 'span': {'begin': 19, 'end': 27}}, {'obj': 'Species:9838', 'span': {'begin': 390, 'end': 395}}, {'obj': 'Species:1335626', 'span': {'begin': 483, 'end': 491}}, {'obj': 'Species:1335626', 'span': {'begin': 522, 'end': 530}}, {'obj': 'Species:1335626', 'span': {'begin': 614, 'end': 622}}, {'obj': 'Species:1335626', 'span': {'begin': 727, 'end': 735}}]}]\n",
      "[{'sourceid': '25856575', 'sourcedb': 'PubMed', 'text': 'Molecular typing and epidemiology profiles of human adenovirus infection among paediatric patients with severe acute respiratory infection in China. BACKGROUND: Human adenoviruses (HAdVs) have been recognised as pathogens that cause a broad spectrum of diseases. The studies on HAdV infection among children with severe acute respiratory infection (SARI) are limited. OBJECTIVE: To investigate the prevalence, epidemiology, and genotype of HAdV among children with SARI in China. STUDY DESIGN: Nasopharyngeal aspirates (NPAs) or induced sputum (IS) was collected from hospitalised children with SARIs in Beijing (representing Northern China; n = 259) and Zhejiang Province (representing Eastern China; n = 293) from 2007 to 2010. The prevalence of HAdV was screened by polymerase chain reaction (PCR), followed by sequence typing of PCR fragments that targeted the second half of the hexon gene. In addition, co-infection with other human respiratory viruses, related epidemiological profiles and clinical presentations were investigated. RESULTS AND CONCLUSIONS: In total, 76 (13.8%) of 552 SARI patients were positive for HAdV, and the infection rates of HAdV in Northern and Eastern China were 20.1% (n = 52) and 8.2% (n = 24), respectively. HAdV co-infection with other respiratory viruses was frequent (infection rates: Northern China, 90.4%; Eastern China, 70.8%). The peak seasons for HAdV-B infection was winter and spring. Additionally, members of multiple species (Human mastadenovirus B, C, D and E) were circulating among paediatric patients with SARI, of which HAdV-B (34/52; 65.4%) and HAdV-C (20/24, 83.3%) were the most predominant in Northern and Eastern China, respectively. These findings provide a benchmark for future epidemiology and prevention strategies for HAdV.', 'denotations': [{'obj': 'Disease:D000257', 'span': {'begin': 52, 'end': 72}}, {'obj': 'Disease:D045169', 'span': {'begin': 104, 'end': 138}}, {'obj': 'Disease:D007239', 'span': {'begin': 278, 'end': 292}}, {'obj': 'Disease:D045169', 'span': {'begin': 313, 'end': 347}}, {'obj': 'Disease:D045169', 'span': {'begin': 349, 'end': 353}}, {'obj': 'Disease:D045169', 'span': {'begin': 465, 'end': 469}}, {'obj': 'Disease:D060085', 'span': {'begin': 909, 'end': 921}}, {'obj': 'Disease:D045169', 'span': {'begin': 1092, 'end': 1096}}, {'obj': 'Disease:D060085', 'span': {'begin': 1245, 'end': 1262}}, {'obj': 'Disease:D045169', 'span': {'begin': 1559, 'end': 1563}}, {'obj': 'Chemical:', 'span': {'begin': 520, 'end': 524}}, {'obj': 'Chemical:', 'span': {'begin': 494, 'end': 518}}, {'obj': 'Species:9606', 'span': {'begin': 161, 'end': 166}}, {'obj': 'Species:9606', 'span': {'begin': 46, 'end': 51}}, {'obj': 'Species:9606', 'span': {'begin': 90, 'end': 98}}, {'obj': 'Species:9606', 'span': {'begin': 299, 'end': 307}}, {'obj': 'Species:9606', 'span': {'begin': 451, 'end': 459}}, {'obj': 'Species:9606', 'span': {'begin': 581, 'end': 589}}, {'obj': 'Species:9606', 'span': {'begin': 933, 'end': 938}}, {'obj': 'Species:10239', 'span': {'begin': 951, 'end': 958}}, {'obj': 'Species:9606', 'span': {'begin': 1097, 'end': 1105}}, {'obj': 'Species:10239', 'span': {'begin': 1286, 'end': 1293}}, {'obj': 'Species:108098', 'span': {'begin': 1392, 'end': 1398}}, {'obj': 'Species:108098', 'span': {'begin': 1475, 'end': 1497}}, {'obj': 'Species:9606', 'span': {'begin': 1545, 'end': 1553}}, {'obj': 'Species:108098', 'span': {'begin': 1574, 'end': 1580}}, {'obj': 'Species:129951', 'span': {'begin': 1600, 'end': 1606}}]}]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'sourceid': '25948687', 'sourcedb': 'PubMed', 'text': \"Epidemic predictions in an imperfect world: modelling disease spread with partial data. 'Big-data' epidemic models are being increasingly used to influence government policy to help with control and eradication of infectious diseases. In the case of livestock, detailed movement records have been used to parametrize realistic transmission models. While livestock movement data are readily available in the UK and other countries in the EU, in many countries around the world, such detailed data are not available. By using a comprehensive database of the UK cattle trade network, we implement various sampling strategies to determine the quantity of network data required to give accurate epidemiological predictions. It is found that by targeting nodes with the highest number of movements, accurate predictions on the size and spatial spread of epidemics can be made. This work has implications for countries such as the USA, where access to data is limited, and developing countries that may lack the resources to collect a full dataset on livestock movements.\", 'denotations': [{'obj': 'Disease:D003141', 'span': {'begin': 214, 'end': 233}}, {'obj': 'Species:9913', 'span': {'begin': 559, 'end': 565}}]}]\n",
      "[{'sourceid': '26041555', 'sourcedb': 'PubMed', 'text': 'LeishVet update and recommendations on feline leishmaniosis. Limited data is available on feline leishmaniosis (FeL) caused by Leishmania infantum worldwide. The LeishVet group presents in this report a review of the current knowledge on FeL, the epidemiological role of the cat in L. infantum infection, clinical manifestations, and recommendations on diagnosis, treatment and monitoring, prognosis and prevention of infection, in order to standardize the management of this disease in cats. The consensus of opinions and recommendations was formulated by combining a comprehensive review of evidence-based studies and case reports, clinical experience and critical consensus discussions. While subclinical feline infections are common in areas endemic for canine leishmaniosis, clinical illness due to L. infantum in cats is rare. The prevalence rates of feline infection with L. infantum in serological or molecular-based surveys range from 0% to more than 60%. Cats are able to infect sand flies and, therefore, they may act as a secondary reservoir, with dogs being the primary natural reservoir. The most common clinical signs and clinicopathological abnormalities compatible with FeL include lymph node enlargement and skin lesions such as ulcerative, exfoliative, crusting or nodular dermatitis (mainly on the head or distal limbs), ocular lesions (mainly uveitis), feline chronic gingivostomatitis syndrome, mucocutaneous ulcerative or nodular lesions, hypergammaglobulinaemia and mild normocytic normochromic anaemia. Clinical illness is frequently associated with impaired immunocompetence, as in case of retroviral coinfections or immunosuppressive therapy. Diagnosis is based on serology, polymerase chain reaction (PCR), cytology, histology, immunohistochemistry (IHC) or culture. If serological testing is negative or low positive in a cat with clinical signs compatible with FeL, the diagnosis of leishmaniosis should not be excluded and additional diagnostic methods (cytology, histology with IHC, PCR, culture) should be employed. The most common treatment used is allopurinol. Meglumine antimoniate has been administered in very few reported cases. Both drugs are administered alone and most cats recover clinically after therapy. Follow-up of treated cats with routine laboratory tests, serology and PCR is essential for prevention of clinical relapses. Specific preventative measures for this infection in cats are currently not available.', 'denotations': [{'obj': 'Disease:D005767', 'span': {'begin': 127, 'end': 146}}, {'obj': 'Disease:D007239', 'span': {'begin': 282, 'end': 303}}, {'obj': 'Disease:D007239', 'span': {'begin': 418, 'end': 427}}, {'obj': 'Disease:D007239', 'span': {'begin': 715, 'end': 725}}, {'obj': 'Disease:D013568', 'span': {'begin': 780, 'end': 796}}, {'obj': 'Disease:D005767', 'span': {'begin': 804, 'end': 815}}, {'obj': 'Disease:D007239', 'span': {'begin': 864, 'end': 873}}, {'obj': 'Disease:D005767', 'span': {'begin': 879, 'end': 890}}, {'obj': 'Disease:D018376', 'span': {'begin': 1137, 'end': 1170}}, {'obj': 'Disease:D014388', 'span': {'begin': 1199, 'end': 1221}}, {'obj': 'Disease:D012871', 'span': {'begin': 1226, 'end': 1238}}, {'obj': 'Disease:D003872', 'span': {'begin': 1284, 'end': 1302}}, {'obj': 'Disease:D015821', 'span': {'begin': 1341, 'end': 1355}}, {'obj': 'Disease:D014605', 'span': {'begin': 1364, 'end': 1371}}, {'obj': 'Disease:D002908', 'span': {'begin': 1381, 'end': 1415}}, {'obj': 'Disease:D014456', 'span': {'begin': 1431, 'end': 1460}}, {'obj': 'Disease:D000740', 'span': {'begin': 1519, 'end': 1526}}, {'obj': 'Disease:D060085', 'span': {'begin': 1616, 'end': 1639}}, {'obj': 'Disease:D007239', 'span': {'begin': 2414, 'end': 2423}}, {'obj': 'Chemical:C005278', 'span': {'begin': 2096, 'end': 2117}}, {'obj': 'Chemical:D000493', 'span': {'begin': 2083, 'end': 2094}}, {'obj': 'Species:9685', 'span': {'begin': 487, 'end': 491}}, {'obj': 'Species:9615', 'span': {'begin': 758, 'end': 764}}, {'obj': 'Species:9685', 'span': {'begin': 819, 'end': 823}}, {'obj': 'Species:9685', 'span': {'begin': 965, 'end': 969}}, {'obj': 'Species:9615', 'span': {'begin': 1060, 'end': 1064}}, {'obj': 'Species:9685', 'span': {'begin': 2211, 'end': 2215}}, {'obj': 'Species:9685', 'span': {'begin': 2271, 'end': 2275}}, {'obj': 'Species:9685', 'span': {'begin': 2427, 'end': 2431}}]}]\n",
      "[{'sourceid': '26325027', 'sourcedb': 'PubMed', 'text': 'Cullin4 Is Pro-Viral during West Nile Virus Infection of Culex Mosquitoes. Although mosquitoes serve as vectors of many pathogens of public health importance, their response to viral infection is poorly understood. It also remains to be investigated whether viruses deploy some mechanism to be able to overcome this immune response. Here, we have used an RNA-Seq approach to identify differentially regulated genes in Culex quinquefasciatus cells following West Nile virus (WNV) infection, identifying 265 transcripts from various cellular pathways that were either upregulated or downregulated. Ubiquitin-proteasomal pathway genes, comprising 12% of total differentially regulated genes, were selected for further validation by real time RT-qPCR and functional analysis. It was found that treatment of infected cells with proteasomal inhibitor, MG-132, decreased WNV titers, indicating importance of this pathway during infection process. In infection models, the Culex ortholog of mammalian Cul4A/B (cullin RING ubiquitin ligase) was found to be upregulated in vitro as well as in vivo, especially in midguts of mosquitoes. Gene knockdown using dsRNA and overexpression studies indicated that Culex Cul4 acts as a pro-viral protein by degradation of CxSTAT via ubiquitin-proteasomal pathway. We also show that gene knockdown of Culex Cul4 leads to activation of the Jak-STAT pathway in mosquitoes leading to decrease viral replication in the body as well as saliva. Our results suggest a novel mechanism adopted by WNV to overcome mosquito immune response and increase viral replication.', 'denotations': [{'obj': 'Disease:D014777', 'span': {'begin': 177, 'end': 192}}, {'obj': 'Disease:D007239', 'span': {'begin': 479, 'end': 488}}, {'obj': 'Disease:D007239', 'span': {'begin': 921, 'end': 930}}, {'obj': 'Disease:D007239', 'span': {'begin': 943, 'end': 952}}, {'obj': 'Chemical:C072553', 'span': {'begin': 846, 'end': 852}}, {'obj': 'Gene:8451;8450', 'span': {'begin': 993, 'end': 1000}}, {'obj': 'Species:11082', 'span': {'begin': 28, 'end': 43}}, {'obj': 'Species:10239', 'span': {'begin': 258, 'end': 265}}, {'obj': 'Species:7176', 'span': {'begin': 418, 'end': 440}}, {'obj': 'Species:11082', 'span': {'begin': 457, 'end': 472}}, {'obj': 'Species:11082', 'span': {'begin': 474, 'end': 477}}, {'obj': 'Species:11082', 'span': {'begin': 864, 'end': 867}}, {'obj': 'Species:9606', 'span': {'begin': 983, 'end': 992}}, {'obj': 'Species:11082', 'span': {'begin': 1517, 'end': 1520}}]}]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'sourceid': '26266418', 'sourcedb': 'PubMed', 'text': 'eEF1A Interacts with the NS5A Protein and Inhibits the Growth of Classical Swine Fever Virus. The NS5A protein of classical swine fever virus (CSFV) is involved in the RNA synthesis and viral replication. However, the NS5A-interacting cellular proteins engaged in the CSFV replication are poorly defined. Using yeast two-hybrid screen, the eukaryotic elongation factor 1A (eEF1A) was identified to be an NS5A-binding partner. The NS5A-eEF1A interaction was confirmed by coimmunoprecipitation, glutathione S-transferase (GST) pulldown and laser confocal microscopy assays. The domain I of eEF1A was shown to be critical for the NS5A-eEF1A interaction. Overexpression of eEF1A suppressed the CSFV growth markedly, and conversely, knockdown of eEF1A enhanced the CSFV replication significantly. Furthermore, eEF1A, as well as NS5A, was found to reduce the translation efficiency of the internal ribosome entry site (IRES) of CSFV in a dose-dependent manner, as demonstrated by luciferase reporter assay. Streptavidin pulldown assay revealed that eEF1A could bind to the CSFV IRES. Collectively, our results suggest that eEF1A interacts with NS5A and negatively regulates the growth of CSFV.', 'denotations': [{'obj': 'Disease:D006691', 'span': {'begin': 114, 'end': 141}}, {'obj': 'Disease:D006691', 'span': {'begin': 143, 'end': 147}}, {'obj': 'Disease:D006691', 'span': {'begin': 65, 'end': 92}}, {'obj': 'Disease:D006691', 'span': {'begin': 268, 'end': 272}}, {'obj': 'Disease:D006691', 'span': {'begin': 690, 'end': 694}}, {'obj': 'Disease:D006691', 'span': {'begin': 760, 'end': 764}}, {'obj': 'Disease:D006691', 'span': {'begin': 922, 'end': 926}}, {'obj': 'Disease:D006691', 'span': {'begin': 1067, 'end': 1071}}, {'obj': 'Disease:D006691', 'span': {'begin': 1182, 'end': 1186}}, {'obj': 'Chemical:', 'span': {'begin': 493, 'end': 506}}, {'obj': 'Gene:1915', 'span': {'begin': 0, 'end': 5}}, {'obj': 'Gene:1915', 'span': {'begin': 340, 'end': 371}}, {'obj': 'Gene:1915', 'span': {'begin': 373, 'end': 378}}, {'obj': 'Gene:1915', 'span': {'begin': 435, 'end': 440}}, {'obj': 'Gene:1915', 'span': {'begin': 588, 'end': 593}}, {'obj': 'Gene:1915', 'span': {'begin': 632, 'end': 637}}, {'obj': 'Gene:1915', 'span': {'begin': 669, 'end': 674}}, {'obj': 'Gene:1915', 'span': {'begin': 741, 'end': 746}}, {'obj': 'Gene:1915', 'span': {'begin': 805, 'end': 810}}, {'obj': 'Gene:1915', 'span': {'begin': 1043, 'end': 1048}}, {'obj': 'Gene:1915', 'span': {'begin': 1117, 'end': 1122}}]}]\n",
      "[{'sourceid': '26678835', 'sourcedb': 'PubMed', 'text': 'Continued high incidence of children with severe influenza A(H1N1)pdm09 admitted to paediatric intensive care units in Germany during the first three post-pandemic influenza seasons, 2010/11-2012/13. BACKGROUND: Previous influenza surveillance at paediatric intensive care units (PICUs) in Germany indicated increased incidence of PICU admissions for the pandemic influenza subtype A(H1N1)pdm09. We investigated incidence and clinical characteristics of influenza in children admitted to PICUs during the first three post-pandemic influenza seasons, using active screening. METHODS: We conducted a prospective surveillance study in 24 PICUs in Bavaria (Germany) from October 2010 to September 2013. Influenza cases among children between 1  month and 16  years of age admitted to these PICUs with acute respiratory infection were confirmed by PCR analysis of respiratory secretions. RESULTS: A total of 24/7/20 influenza-associated PICU admissions were recorded in the post-pandemic seasons 1/2/3; incidence estimates per 100,000 children were 1.72/0.76/1.80, respectively. Of all 51 patients, 80  % had influenza A, including 65  % with A(H1N1)pdm09. Influenza A(H1N1)pdm09 was almost absent in season 2 (incidence 0.11), but dominated PICU admissions in seasons 1 (incidence 1.35) and 3 (incidence 1.17). Clinical data was available for 47 influenza patients; median age was 4.8  years (IQR 1.6-11.0). The most frequent diagnoses were influenza-associated pneumonia (62  %), bronchitis/bronchiolitis (32  %), secondary bacterial pneumonia (26  %), and ARDS (21  %). Thirty-six patients (77  %) had underlying medical conditions. Median duration of PICU stay was 3  days (IQR 1-11). Forty-seven per cent of patients received mechanical ventilation, and one patient (2  %) extracorporeal membrane oxygenation; 19  % were treated with oseltamivir. Five children (11  %) had pulmonary sequelae. Five children (11  %) died; all had underlying chronic conditions and were infected with A(H1N1)pdm09. In season 3, patients with A(H1N1)pdm09 were younger than in season 1 (p   =   0.020), were diagnosed more often with bronchitis/bronchiolitis (p   =   0.004), and were admitted to a PICU later after the onset of influenza symptoms (p   =   0.041). CONCLUSIONS: Active screening showed a continued high incidence of A(H1N1)pdm09-associated PICU admissions in the post-pandemic seasons 1 and 3, and indicated possible underestimation of incidence in previous German studies. The age shift of severe A(H1N1)pdm09 towards younger children may be explained by increasing immunity in the older paediatric population. The high proportion of patients with underlying chronic conditions indicates the importance of consistent implementation of the current influenza vaccination recommendations for risk groups in Germany.', 'denotations': [{'obj': 'Disease:D007251', 'span': {'begin': 221, 'end': 230}}, {'obj': 'Disease:D007251', 'span': {'begin': 49, 'end': 60}}, {'obj': 'Disease:D007251', 'span': {'begin': 164, 'end': 173}}, {'obj': 'Disease:D007251', 'span': {'begin': 364, 'end': 373}}, {'obj': 'Disease:D007251', 'span': {'begin': 454, 'end': 463}}, {'obj': 'Disease:D007251', 'span': {'begin': 531, 'end': 540}}, {'obj': 'Disease:D012141', 'span': {'begin': 797, 'end': 824}}, {'obj': 'Disease:D007251', 'span': {'begin': 911, 'end': 920}}, {'obj': 'Disease:D007251', 'span': {'begin': 1104, 'end': 1115}}, {'obj': 'Disease:D007251', 'span': {'begin': 1152, 'end': 1163}}, {'obj': 'Disease:D007251', 'span': {'begin': 1342, 'end': 1351}}, {'obj': 'Disease:D007251', 'span': {'begin': 1437, 'end': 1446}}, {'obj': 'Disease:D011014', 'span': {'begin': 1458, 'end': 1467}}, {'obj': 'Disease:D001991', 'span': {'begin': 1477, 'end': 1487}}, {'obj': 'Disease:D001988', 'span': {'begin': 1488, 'end': 1501}}, {'obj': 'Disease:D011014', 'span': {'begin': 1531, 'end': 1540}}, {'obj': 'Disease:D012128', 'span': {'begin': 1554, 'end': 1558}}, {'obj': 'Disease:D001991', 'span': {'begin': 2114, 'end': 2124}}, {'obj': 'Disease:D001988', 'span': {'begin': 2125, 'end': 2138}}, {'obj': 'Disease:D007251', 'span': {'begin': 2209, 'end': 2218}}, {'obj': 'Disease:D007251', 'span': {'begin': 2744, 'end': 2753}}, {'obj': 'Chemical:D053139', 'span': {'begin': 1834, 'end': 1845}}, {'obj': 'Species:9606', 'span': {'begin': 28, 'end': 36}}, {'obj': 'Species:9606', 'span': {'begin': 467, 'end': 475}}, {'obj': 'Species:9606', 'span': {'begin': 721, 'end': 729}}, {'obj': 'Species:9606', 'span': {'begin': 1030, 'end': 1038}}, {'obj': 'Species:9606', 'span': {'begin': 1084, 'end': 1092}}, {'obj': 'Species:9606', 'span': {'begin': 1352, 'end': 1360}}, {'obj': 'Species:9606', 'span': {'begin': 1579, 'end': 1587}}, {'obj': 'Species:9606', 'span': {'begin': 1708, 'end': 1716}}, {'obj': 'Species:9606', 'span': {'begin': 1758, 'end': 1765}}, {'obj': 'Species:9606', 'span': {'begin': 1852, 'end': 1860}}, {'obj': 'Species:9606', 'span': {'begin': 1898, 'end': 1906}}, {'obj': 'Species:9606', 'span': {'begin': 2009, 'end': 2017}}, {'obj': 'Species:9606', 'span': {'begin': 2523, 'end': 2531}}, {'obj': 'Species:9606', 'span': {'begin': 2631, 'end': 2639}}]}]\n",
      "[{'sourceid': '26696249', 'sourcedb': 'PubMed', 'text': 'Etiology and Factors Associated with Pneumonia in Children under 5 Years of Age in Mali: A Prospective Case-Control Study. BACKGROUND: There are very limited data on children with pneumonia in Mali. The objective was to assess the etiology and factors associated with community-acquired pneumonia in hospitalized children <5 years of age in Mali. METHODS: A prospective hospital-based case-control study was implemented in the Pediatric department of Gabriel Tour   University Hospital at Bamako, Mali, between July 2011-December 2012. Cases were children with radiologically-confirmed pneumonia; Controls were hospitalized children without respiratory features, matched for age and period. Respiratory specimens, were collected to identify 19 viruses and 5 bacteria. Whole blood was collected from cases only. Factors associated with pneumonia were assessed by multivariate logistic regression. RESULTS: Overall, 118 cases and 98 controls were analyzed; 44.1% were female, median age was 11 months. Among pneumonia cases, 30.5% were hypoxemic at admission, mortality was 4.2%. Pneumonia cases differed from the controls regarding clinical signs and symptoms but not in terms of past medical history. Multivariate analysis of nasal swab findings disclosed that S. pneumoniae (adjusted odds ratio [aOR] = 3.4, 95% confidence interval [95% CI]: 1.6-7.0), human metapneumovirus (aOR = 17.2, 95% CI: 2.0-151.4), respiratory syncytial virus [RSV] (aOR = 7.4, 95% CI: 2.3-23.3), and influenza A virus (aOR = 10.7, 95% CI: 1.0-112.2) were associated with pneumonia, independently of patient age, gender, period, and other pathogens. Distribution of S. pneumoniae and RSV differed by season with higher rates of S. pneumoniae in January-June and of RSV in July-September. Pneumococcal serotypes 1 and 5 were more frequent in pneumonia cases than in the controls (P = 0.009, and P = 0.04, respectively). CONCLUSIONS: In this non-PCV population from Mali, pneumonia in children was mainly attributed to S. pneumoniae, RSV, human metapneumovirus, and influenza A virus. Increased pneumococcal conjugate vaccine coverage in children could significantly reduce the burden of pneumonia in sub-Saharan African countries.', 'denotations': [{'obj': 'Disease:D011014', 'span': {'begin': 37, 'end': 46}}, {'obj': 'Disease:D011014', 'span': {'begin': 180, 'end': 189}}, {'obj': 'Disease:D011014', 'span': {'begin': 287, 'end': 296}}, {'obj': 'Disease:D011014', 'span': {'begin': 586, 'end': 595}}, {'obj': 'Disease:D011014', 'span': {'begin': 835, 'end': 844}}, {'obj': 'Disease:D011014', 'span': {'begin': 1006, 'end': 1015}}, {'obj': 'Disease:D011014', 'span': {'begin': 1078, 'end': 1087}}, {'obj': 'Disease:D011014', 'span': {'begin': 1264, 'end': 1274}}, {'obj': 'Disease:D018357', 'span': {'begin': 1408, 'end': 1435}}, {'obj': 'Disease:D018357', 'span': {'begin': 1437, 'end': 1440}}, {'obj': 'Disease:D015658', 'span': {'begin': 1477, 'end': 1494}}, {'obj': 'Disease:D011014', 'span': {'begin': 1548, 'end': 1557}}, {'obj': 'Disease:D011014', 'span': {'begin': 1645, 'end': 1655}}, {'obj': 'Disease:D018357', 'span': {'begin': 1660, 'end': 1663}}, {'obj': 'Disease:D011014', 'span': {'begin': 1707, 'end': 1717}}, {'obj': 'Disease:D018357', 'span': {'begin': 1741, 'end': 1744}}, {'obj': 'Disease:D011014', 'span': {'begin': 1817, 'end': 1826}}, {'obj': 'Disease:D011014', 'span': {'begin': 1946, 'end': 1955}}, {'obj': 'Disease:D011014', 'span': {'begin': 1996, 'end': 2006}}, {'obj': 'Disease:D018357', 'span': {'begin': 2008, 'end': 2011}}, {'obj': 'Disease:D015658', 'span': {'begin': 2040, 'end': 2057}}, {'obj': 'Disease:D011014', 'span': {'begin': 2162, 'end': 2171}}, {'obj': 'Species:9606', 'span': {'begin': 166, 'end': 174}}, {'obj': 'Species:9606', 'span': {'begin': 50, 'end': 58}}, {'obj': 'Species:9606', 'span': {'begin': 313, 'end': 321}}, {'obj': 'Species:9606', 'span': {'begin': 547, 'end': 555}}, {'obj': 'Species:9606', 'span': {'begin': 624, 'end': 632}}, {'obj': 'Species:10239', 'span': {'begin': 744, 'end': 751}}, {'obj': 'Species:2', 'span': {'begin': 758, 'end': 766}}, {'obj': 'Species:162145', 'span': {'begin': 1353, 'end': 1374}}, {'obj': 'Species:9606', 'span': {'begin': 1576, 'end': 1583}}, {'obj': 'Species:158372', 'span': {'begin': 1920, 'end': 1923}}, {'obj': 'Species:9606', 'span': {'begin': 1959, 'end': 1967}}, {'obj': 'Species:162145', 'span': {'begin': 2013, 'end': 2034}}, {'obj': 'Species:9606', 'span': {'begin': 2112, 'end': 2120}}]}]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'sourceid': '26684827', 'sourcedb': 'PubMed', 'text': 'Prostaglandin E2 stimulates normal bronchial epithelial cell growth through induction of c-Jun and PDK1, a kinase implicated in oncogenesis. BACKGROUND: Cyclooxygenase-2-derived prostaglandin E2 (PGE2), a bioactive eicosanoid, has been implicated in many biological processes including reproduction, inflammation and tumor growth. We previously showed that PGE2 stimulated lung cancer cell growth and progression through PGE2 receptor EP2/EP4-mediated kinase signaling pathways. However, the role of PGE2 in controlling lung airway epithelial cell phenotype remains unknown. We evaluated the effects of c-Jun and 3-phosphoinositede dependent protein kinase-1 (PDK1) in mediating epithelial cell hyperplasia induced by PGE2. METHOD: The bronchial epithelial cell lines BEAS-2B and HBEc14-KT were cultured and then treated with PGE2. PDK1 small interfering RNA (siRNA) and a PDK1 inhibitor, an antagonist of the PGE2 receptor subtype EP4 and EP4 siRNA, c-Jun siRNA, and overexpressions of c-Jun and PDK1 have been used to evaluate the effects on cell proliferation. RESULTS: We demonstrated that PGE2 increased normal bronchial epithelial cell proliferation through induction of PDK1, an ankyrin repeat-containing Ser/Thr kinase implicated in the induction of apoptosis and the suppression of tumor growth. PDK1 siRNA and a PDK1 inhibitor blocked the effects of PGE2 on normal cell growth. The PGE2-induced PDK1 expression was blocked by an antagonist of the PGE2 receptor subtype EP4 and by EP4 siRNA. In addition, we showed that induction of PDK1 by PGE2 was associated with induction of the transcription factor, c-Jun protein. Silencing of c-Jun using siRNA and point mutations of c-Jun sites in the PDK1 gene promoter resulted in blockade of PDK1 expression and promoter activity induced by PGE2. In contrast, overexpression of c-Jun induced PDK1 gene promoter activity and expression followed increased cell proliferation. CONCLUSION: PGE2 increases normal bronchial epithelial cell proliferation through increased PDK1 gene expression that is dependent on EP4 and induction of c-Jun. Therewith, our data suggest a new role of c-Jun and PDK1 in mediating epithelial cell hyperplasia induced by PGE2.', 'denotations': [{'obj': 'Disease:D007249', 'span': {'begin': 300, 'end': 312}}, {'obj': 'Disease:D009369', 'span': {'begin': 317, 'end': 322}}, {'obj': 'Disease:D008175', 'span': {'begin': 373, 'end': 384}}, {'obj': 'Disease:D006965', 'span': {'begin': 695, 'end': 706}}, {'obj': 'Disease:D009369', 'span': {'begin': 1291, 'end': 1296}}, {'obj': 'Disease:D006965', 'span': {'begin': 2175, 'end': 2186}}, {'obj': 'Chemical:D015232', 'span': {'begin': 2198, 'end': 2202}}, {'obj': 'Chemical:D015232', 'span': {'begin': 1939, 'end': 1943}}, {'obj': 'Chemical:D015232', 'span': {'begin': 1794, 'end': 1798}}, {'obj': 'Chemical:D015232', 'span': {'begin': 1550, 'end': 1554}}, {'obj': 'Chemical:D015232', 'span': {'begin': 1457, 'end': 1461}}, {'obj': 'Chemical:D015232', 'span': {'begin': 1392, 'end': 1396}}, {'obj': 'Chemical:D015232', 'span': {'begin': 1360, 'end': 1364}}, {'obj': 'Chemical:D015232', 'span': {'begin': 1094, 'end': 1098}}, {'obj': 'Chemical:D015232', 'span': {'begin': 910, 'end': 914}}, {'obj': 'Chemical:D015232', 'span': {'begin': 826, 'end': 830}}, {'obj': 'Chemical:D015232', 'span': {'begin': 718, 'end': 722}}, {'obj': 'Chemical:D015232', 'span': {'begin': 500, 'end': 504}}, {'obj': 'Chemical:D015232', 'span': {'begin': 421, 'end': 425}}, {'obj': 'Chemical:D015232', 'span': {'begin': 357, 'end': 361}}, {'obj': 'Chemical:CHEBI:23899', 'span': {'begin': 215, 'end': 225}}, {'obj': 'Chemical:D015232', 'span': {'begin': 196, 'end': 200}}, {'obj': 'Chemical:D015232', 'span': {'begin': 178, 'end': 194}}, {'obj': 'Chemical:D015232', 'span': {'begin': 0, 'end': 16}}, {'obj': 'Gene:5743', 'span': {'begin': 153, 'end': 169}}, {'obj': 'Gene:3725', 'span': {'begin': 89, 'end': 94}}, {'obj': 'Gene:5163', 'span': {'begin': 99, 'end': 103}}, {'obj': 'Gene:5732', 'span': {'begin': 435, 'end': 438}}, {'obj': 'Gene:5734', 'span': {'begin': 439, 'end': 442}}, {'obj': 'Gene:3725', 'span': {'begin': 603, 'end': 608}}, {'obj': 'Gene:5163', 'span': {'begin': 660, 'end': 664}}, {'obj': 'Gene:5163', 'span': {'begin': 832, 'end': 836}}, {'obj': 'Gene:5163', 'span': {'begin': 873, 'end': 877}}, {'obj': 'Gene:5734', 'span': {'begin': 932, 'end': 935}}, {'obj': 'Gene:5734', 'span': {'begin': 940, 'end': 943}}, {'obj': 'Gene:3725', 'span': {'begin': 951, 'end': 956}}, {'obj': 'Gene:3725', 'span': {'begin': 987, 'end': 992}}, {'obj': 'Gene:5163', 'span': {'begin': 997, 'end': 1001}}, {'obj': 'Gene:5163', 'span': {'begin': 1177, 'end': 1181}}, {'obj': 'Gene:5163', 'span': {'begin': 1305, 'end': 1309}}, {'obj': 'Gene:5163', 'span': {'begin': 1322, 'end': 1326}}, {'obj': 'Gene:5163', 'span': {'begin': 1405, 'end': 1409}}, {'obj': 'Gene:5734', 'span': {'begin': 1479, 'end': 1482}}, {'obj': 'Gene:5734', 'span': {'begin': 1490, 'end': 1493}}, {'obj': 'Gene:5163', 'span': {'begin': 1542, 'end': 1546}}, {'obj': 'Gene:3725', 'span': {'begin': 1614, 'end': 1619}}, {'obj': 'Gene:3725', 'span': {'begin': 1642, 'end': 1647}}, {'obj': 'Gene:3725', 'span': {'begin': 1683, 'end': 1688}}, {'obj': 'Gene:5163', 'span': {'begin': 1702, 'end': 1706}}, {'obj': 'Gene:5163', 'span': {'begin': 1745, 'end': 1749}}, {'obj': 'Gene:3725', 'span': {'begin': 1831, 'end': 1836}}, {'obj': 'Gene:5163', 'span': {'begin': 1845, 'end': 1849}}, {'obj': 'Gene:5163', 'span': {'begin': 2019, 'end': 2023}}, {'obj': 'Gene:5734', 'span': {'begin': 2061, 'end': 2064}}, {'obj': 'Gene:3725', 'span': {'begin': 2082, 'end': 2087}}, {'obj': 'Gene:3725', 'span': {'begin': 2131, 'end': 2136}}, {'obj': 'Gene:5163', 'span': {'begin': 2141, 'end': 2145}}]}]\n",
      "[{'sourceid': '26884414', 'sourcedb': 'PubMed', 'text': 'The role of respiratory viruses in the etiology of bacterial pneumonia: An ecological perspective. Pneumonia is the leading cause of death among children less than 5 years old worldwide. A wide range of viral, bacterial and fungal agents can cause pneumonia: although viruses are the most common etiologic agent, the severity of clinical symptoms associated with bacterial pneumonia and increasing antibiotic resistance makes bacterial pneumonia a major public health concern. Bacterial pneumonia can follow upper respiratory viral infection and complicate lower respiratory viral infection. Secondary bacterial pneumonia is a major cause of influenza-related deaths. In this review, we evaluate the following hypotheses: (i) respiratory viruses influence the etiology of pneumonia by altering bacterial community structure in the upper respiratory tract (URT) and (ii) respiratory viruses promote or inhibit colonization of the lower respiratory tract (LRT) by certain bacterial species residing in the URT. We conducted a systematic review of the literature to examine temporal associations between respiratory viruses and bacteria and a targeted review to identify potential mechanisms of interactions. We conclude that viruses both alter the bacterial community in the URT and promote bacterial colonization of the LRT. However, it is uncertain whether changes in the URT bacterial community play a substantial role in pneumonia etiology. The exception is Streptococcus pneumoniae where a strong link between viral co-infection, increased carriage and pneumococcal pneumonia has been established.', 'denotations': [{'obj': 'Disease:D011014', 'span': {'begin': 99, 'end': 108}}, {'obj': 'Disease:D003643', 'span': {'begin': 133, 'end': 138}}, {'obj': 'Disease:D018410', 'span': {'begin': 51, 'end': 70}}, {'obj': 'Disease:D011014', 'span': {'begin': 248, 'end': 257}}, {'obj': 'Disease:D018410', 'span': {'begin': 363, 'end': 382}}, {'obj': 'Disease:D018410', 'span': {'begin': 426, 'end': 445}}, {'obj': 'Disease:D018410', 'span': {'begin': 477, 'end': 496}}, {'obj': 'Disease:D012141', 'span': {'begin': 501, 'end': 590}}, {'obj': 'Disease:D018410', 'span': {'begin': 592, 'end': 621}}, {'obj': 'Disease:D003643', 'span': {'begin': 660, 'end': 666}}, {'obj': 'Disease:D011014', 'span': {'begin': 772, 'end': 781}}, {'obj': 'Disease:D011014', 'span': {'begin': 1423, 'end': 1432}}, {'obj': 'Disease:D014777', 'span': {'begin': 1513, 'end': 1531}}, {'obj': 'Disease:D011018', 'span': {'begin': 1556, 'end': 1578}}, {'obj': 'Species:9606', 'span': {'begin': 145, 'end': 153}}, {'obj': 'Species:1313', 'span': {'begin': 1460, 'end': 1484}}]}]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'sourceid': '27094905', 'sourcedb': 'PubMed', 'text': 'Middle East respiratory syndrome coronavirus M protein suppresses type I interferon expression through the inhibition of TBK1-dependent phosphorylation of IRF3. Middle East respiratory syndrome coronavirus (MERS-CoV) infection has claimed hundreds of lives and has become a global threat since its emergence in Saudi Arabia in 2012. The ability of MERS-CoV to evade the host innate antiviral response may contribute to its severe pathogenesis. Many MERS-CoV-encoded proteins were identified to have interferon (IFN)-antagonizing properties, which correlates well with the reduced IFN levels observed in infected patients and ex vivo models. In this study, we fully characterized the IFN-antagonizing property of the MERS-CoV M protein. Expression of MERS-CoV M protein suppressed type I IFN expression in response to Sendai virus infection or poly(I:C) induction. This suppressive effect was found to be specific for the activation of IFN regulatory factor 3 (IRF3) but not nuclear factor-kB. MERS-CoV M protein interacted with TRAF3 and disrupted TRAF3-TBK1 association leading to reduced IRF3 activation. M proteins from MERS-CoV and SARS-CoV have three highly similar conserved N-terminal transmembrane domains and a C-terminal region. Using chimeric and truncation mutants, the N-terminal transmembrane domains of the MERS-CoV M protein were found to be sufficient for its inhibitory effect on IFN expression, whereas the C-terminal domain was unable to induce this suppression. Collectively, our findings suggest a common and conserved mechanism through which highly pathogenic MERS-CoV and SARS-CoV harness their M proteins to suppress type I IFN expression at the level of TBK1-dependent phosphorylation and activation of IRF3 resulting in evasion of the host innate antiviral response.', 'denotations': [{'obj': 'Disease:D015658', 'span': {'begin': 817, 'end': 839}}, {'obj': 'Chemical:', 'span': {'begin': 1426, 'end': 1427}}, {'obj': 'Chemical:', 'span': {'begin': 1282, 'end': 1283}}, {'obj': 'Chemical:', 'span': {'begin': 1220, 'end': 1221}}, {'obj': 'Chemical:', 'span': {'begin': 1181, 'end': 1182}}, {'obj': 'Chemical:', 'span': {'begin': 850, 'end': 851}}, {'obj': 'Species:1335626', 'span': {'begin': 161, 'end': 215}}, {'obj': 'Species:1335626', 'span': {'begin': 0, 'end': 44}}, {'obj': 'Species:1335626', 'span': {'begin': 348, 'end': 356}}, {'obj': 'Species:1335626', 'span': {'begin': 449, 'end': 457}}, {'obj': 'Species:9606', 'span': {'begin': 612, 'end': 620}}, {'obj': 'Species:1335626', 'span': {'begin': 716, 'end': 724}}, {'obj': 'Species:1335626', 'span': {'begin': 750, 'end': 758}}, {'obj': 'Species:1335626', 'span': {'begin': 993, 'end': 1001}}, {'obj': 'Species:1335626', 'span': {'begin': 1123, 'end': 1131}}, {'obj': 'Species:227859', 'span': {'begin': 1136, 'end': 1140}}, {'obj': 'Species:1335626', 'span': {'begin': 1322, 'end': 1330}}, {'obj': 'Species:1335626', 'span': {'begin': 1583, 'end': 1591}}, {'obj': 'Species:227859', 'span': {'begin': 1596, 'end': 1600}}]}]\n",
      "[{'sourceid': '26961383', 'sourcedb': 'PubMed', 'text': 'Reducing burden of disease from residential indoor air exposures in Europe (HEALTHVENT project). BACKGROUND: The annual burden of disease caused indoor air pollution, including polluted outdoor air used to ventilate indoor spaces, is estimated to correspond to a loss of over 2 million healthy life years in the European Union (EU). Based on measurements of the European Environment Agency (EEA), approximately 90  % of EU citizens live in areas where the World Health Organization (WHO) guidelines for air quality of particulate matter sized   <   2.5  mm (PM2.5) are not met. Since sources of pollution reside in both indoor and outdoor air, selecting the most appropriate ventilation strategy is not a simple and straightforward task. METHODS: A framework for developing European health-based ventilation guidelines was created in 2010-2013 in the EU-funded HEALTHVENT project. As a part of the project, the potential efficiency of control policies to health effects caused by residential indoor exposures of fine particulate matter (PM2.5), outdoor bioaerosols, volatile organic compounds (VOC), carbon oxide (CO) radon and dampness was estimated. The analysis was based on scenario comparison, using an outdoor-indoor mass-balance model and varying the ventilation rates. Health effects were estimated with burden of diseases (BoD) calculations taking into account asthma, cardiovascular (CV) diseases, acute toxication, respiratory infections, lung cancer and chronic obstructive pulmonary disease (COPD). RESULTS: The quantitative comparison of three main policy approaches, (i) optimising ventilation rates only; (ii) filtration of outdoor air; and (iii) indoor source control, showed that all three approaches are able to provide substantial reductions in the health risks, varying from approximately 20  % to 44  %, corresponding to 400 000 and 900 000 saved healthy life years in EU-26. PM2.5 caused majority of the health effects in all included countries, but the importance of the other pollutants varied by country. CONCLUSIONS: The present modelling shows, that combination of controlling the indoor air sources and selecting appropriate ventilation rate was the most effective to reduce health risks. If indoor sources cannot be removed or their emissions cannot be limited to an accepted level, ventilation needs to be increased to remove remaining pollutants. In these cases filtration of outdoor air may be needed to prevent increase of health risks.', 'denotations': [{'obj': 'Disease:D004194', 'span': {'begin': 113, 'end': 137}}, {'obj': 'Disease:D004194', 'span': {'begin': 1312, 'end': 1330}}, {'obj': 'Disease:D004194', 'span': {'begin': 1332, 'end': 1335}}, {'obj': 'Disease:D001249', 'span': {'begin': 1370, 'end': 1376}}, {'obj': 'Disease:D002318', 'span': {'begin': 1378, 'end': 1406}}, {'obj': 'Disease:D064420', 'span': {'begin': 1408, 'end': 1424}}, {'obj': 'Disease:D012141', 'span': {'begin': 1426, 'end': 1448}}, {'obj': 'Disease:D008175', 'span': {'begin': 1450, 'end': 1461}}, {'obj': 'Disease:D029424', 'span': {'begin': 1466, 'end': 1503}}, {'obj': 'Disease:D029424', 'span': {'begin': 1505, 'end': 1509}}, {'obj': 'Chemical:CHEBI:23014', 'span': {'begin': 1100, 'end': 1112}}]}]\n",
      "[{'sourceid': '27328634', 'sourcedb': 'PubMed', 'text': 'No involvement of alveolar macrophages in the initiation of carbon nanoparticle induced acute lung inflammation in mice. BACKGROUND: Carbonaceous nanoparticles (CNP) represent a major constituent of urban particulate air pollution, and inhalation of high CNP levels has been described to trigger a pro-inflammatory response of the lung. While several studies identified specific particle characteristics driving respiratory toxicity of low-solubility and low-toxicity particles such as CNP, the major lung cell type, which initiates and drives that response, remains still uncertain. Since alveolar macrophages (AM) are known to effectively phagocytose inhaled particles and play a crucial role for the initiation of pulmonary inflammation caused by invading microbes, we aimed to determine their role for sterile stimuli such as CNP by profiling the primary alveolar cell compartments of the lung. We exposed C57BL/6 mice to 20   g CNP by intratracheal instillation and comprehensively investigated the expression of the underlying mediators during a time span of 3 to 72  h in three different lung cell populations: CD45- (negative) structural cells, CD45+ (positive) leukocytes, and by BAL recovered cells. RESULTS: Bronchoalveolar lavage (BAL) analysis revealed an acute inflammatory response characterized by the most prominent culmination of neutrophil granulocytes from 12 to 24  h after instillation, which declined to basal levels by day 7. As early as 3  h after CNP exposure 50  % of the AM revealed particle laden. BAL concentrations and lung gene expression profiles of TNFa, and the neutrophil chemoattractants CXCL1,-2 and-5 preceded the neutrophil recruitment and showed highest levels after 12  h of CNP exposure, pointing to a significant activation of the inflammation-evoking lung cells at this point of time. AM, isolated from lungs 3 to 12  h after CNP instillation, however, did not show a pro-inflammatory signature. On the contrary, gene expression analysis of different lung cell populations isolated 12  h after CNP instillation revealed CD45-, mainly representing alveolar epithelial type II (ATII) cells as major producer of inflammatory CXCL cytokines. Particularly by CD45- cells expressed Cxcl5 proved to be the most abundant chemokine, being 12  h after CNP exposure 24 (  11) fold induced. CONCLUSION: Our data suggests that AM are noninvolved in the initiation of the inflammatory response. ATII cells, which induced highest CXCL levels early on, might in contrast be the driver of acute neutrophilic inflammation upon pulmonary CNP exposure.', 'denotations': [{'obj': 'Disease:D011014', 'span': {'begin': 88, 'end': 111}}, {'obj': 'Disease:D064420', 'span': {'begin': 412, 'end': 432}}, {'obj': 'Disease:D064420', 'span': {'begin': 459, 'end': 467}}, {'obj': 'Disease:D011014', 'span': {'begin': 717, 'end': 739}}, {'obj': 'Disease:D002282', 'span': {'begin': 2092, 'end': 2119}}, {'obj': 'Disease:D002282', 'span': {'begin': 2121, 'end': 2125}}, {'obj': 'Disease:D002282', 'span': {'begin': 2426, 'end': 2430}}, {'obj': 'Chemical:D002244', 'span': {'begin': 60, 'end': 66}}, {'obj': 'Gene:19264', 'span': {'begin': 1118, 'end': 1122}}, {'obj': 'Gene:19264', 'span': {'begin': 1153, 'end': 1157}}, {'obj': 'Gene:21926', 'span': {'begin': 1583, 'end': 1587}}, {'obj': 'Gene:14825', 'span': {'begin': 1625, 'end': 1639}}, {'obj': 'Gene:19264', 'span': {'begin': 2065, 'end': 2069}}, {'obj': 'Gene:19264', 'span': {'begin': 2199, 'end': 2203}}, {'obj': 'Gene:20311', 'span': {'begin': 2221, 'end': 2226}}, {'obj': 'Species:10090', 'span': {'begin': 115, 'end': 119}}, {'obj': 'Species:10090', 'span': {'begin': 918, 'end': 922}}]}]\n"
     ]
    }
   ],
   "source": [
    "jsonfiles = []\n",
    "\n",
    "for id in pmid:\n",
    "    url = 'https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/'+id+'/PubAnnotation/'\n",
    "    url_list.append(url)\n",
    "    resp = requests.get(url)\n",
    "    content = print(resp.json())\n",
    "    data = jsonfiles.append(content)\n",
    "\n",
    "print(jsonfiles)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'sourceid': '16013892', 'sourcedb': 'PubMed', 'text': 'Appropriate models for the management of infectious diseases. BACKGROUND: Mathematical models have become invaluable management tools for epidemiologists, both shedding light on the mechanisms underlying observed dynamics as well as making quantitative predictions on the effectiveness of different control measures. Here, we explain how substantial biases are introduced by two important, yet largely ignored, assumptions at the core of the vast majority of such models. METHODS AND FINDINGS: First, we use analytical methods to show that (i) ignoring the latent period or (ii) making the common assumption of exponentially distributed latent and infectious periods (when including the latent period) always results in underestimating the basic reproductive ratio of an infection from outbreak data. We then proceed to illustrate these points by fitting epidemic models to data from an influenza outbreak. Finally, we document how such unrealistic a priori assumptions concerning model structure give rise to systematically overoptimistic predictions on the outcome of potential management options. CONCLUSION: This work aims to highlight that, when developing models for public health use, we need to pay careful attention to the intrinsic assumptions embedded within classical frameworks.', 'denotations': [{'obj': 'Disease:D003141', 'span': {'begin': 41, 'end': 60}}, {'obj': 'Disease:D007251', 'span': {'begin': 887, 'end': 896}}]}]\n"
     ]
    }
   ],
   "source": [
    "resp = requests.get('https://www.ncbi.nlm.nih.gov/CBBresearch/Lu/Demo/RESTful/tmTool.cgi/BioConcept/16013892/PubAnnotation/')\n",
    "print(resp.json()) #.content and .text also possible\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
